Julia Version 1.5.0-DEV.107
Commit 248bc460bb (2020-01-19 02:41 UTC)
Platform Info:
  OS: Linux (x86_64-pc-linux-gnu)
  CPU: Intel(R) Xeon(R) Silver 4114 CPU @ 2.20GHz
  WORD_SIZE: 64
  LIBM: libopenlibm
  LLVM: libLLVM-8.0.1 (ORCJIT, skylake)
Environment:
  JULIA_DEPOT_PATH = ::/usr/local/share/julia
  JULIA_NUM_THREADS = 2

 Resolving package versions...
 Installed DataAPI ───────────────────── v1.1.0
 Installed Libiconv_jll ──────────────── v1.16.0+0
 Installed TableTraits ───────────────── v1.0.0
 Installed EzXML ─────────────────────── v1.1.0
 Installed IterTools ─────────────────── v1.3.0
 Installed LibPQ ─────────────────────── v1.0.6
 Installed BinaryProvider ────────────── v0.5.8
 Installed JSON ──────────────────────── v0.21.0
 Installed DataValueInterfaces ───────── v1.0.0
 Installed Parsers ───────────────────── v0.3.10
 Installed XML2_jll ──────────────────── v2.9.9+1
 Installed IteratorInterfaceExtensions ─ v1.0.0
 Installed Mocking ───────────────────── v0.7.0
 Installed TimeZones ─────────────────── v0.10.5
 Installed Syslogs ───────────────────── v0.3.0
 Installed Memento ───────────────────── v0.12.1
 Installed Zlib_jll ──────────────────── v1.2.11+7
 Installed Tables ────────────────────── v0.2.11
 Installed LayerDicts ────────────────── v1.0.0
 Installed Decimals ──────────────────── v0.4.0
 Installed CEnum ─────────────────────── v0.2.0
 Installed OffsetArrays ──────────────── v0.11.4
 Installed DocStringExtensions ───────── v0.8.1
  Updating `~/.julia/environments/v1.5/Project.toml`
  [194296ae] + LibPQ v1.0.6
  Updating `~/.julia/environments/v1.5/Manifest.toml`
  [b99e7846] + BinaryProvider v0.5.8
  [fa961155] + CEnum v0.2.0
  [9a962f9c] + DataAPI v1.1.0
  [e2d170a0] + DataValueInterfaces v1.0.0
  [abce61dc] + Decimals v0.4.0
  [ffbed154] + DocStringExtensions v0.8.1
  [8f5d6c58] + EzXML v1.1.0
  [c8e1da08] + IterTools v1.3.0
  [82899510] + IteratorInterfaceExtensions v1.0.0
  [682c06a0] + JSON v0.21.0
  [6f188dcb] + LayerDicts v1.0.0
  [194296ae] + LibPQ v1.0.6
  [94ce4f54] + Libiconv_jll v1.16.0+0
  [f28f55f0] + Memento v0.12.1
  [78c3b35d] + Mocking v0.7.0
  [6fe1bfb0] + OffsetArrays v0.11.4
  [69de0a69] + Parsers v0.3.10
  [cea106d9] + Syslogs v0.3.0
  [3783bdb8] + TableTraits v1.0.0
  [bd369af6] + Tables v0.2.11
  [f269a46b] + TimeZones v0.10.5
  [02c8fc9c] + XML2_jll v2.9.9+1
  [83775a58] + Zlib_jll v1.2.11+7
  [2a0f44e3] + Base64 
  [ade2ca70] + Dates 
  [8ba89e20] + Distributed 
  [7b1f6079] + FileWatching 
  [b77e0a4c] + InteractiveUtils 
  [76f85450] + LibGit2 
  [8f399da3] + Libdl 
  [37e2e46d] + LinearAlgebra 
  [56ddb016] + Logging 
  [d6f4376e] + Markdown 
  [a63ad114] + Mmap 
  [44cfe95a] + Pkg 
  [de0858da] + Printf 
  [3fa0cd96] + REPL 
  [9a3f8284] + Random 
  [ea8e919c] + SHA 
  [9e88b42a] + Serialization 
  [6462fe0b] + Sockets 
  [8dfed614] + Test 
  [cf7118a7] + UUIDs 
  [4ec0a83e] + Unicode 
  Building TimeZones → `~/.julia/packages/TimeZones/joMm3/deps/build.log`
  Updating `/tmp/jl_BWJpcq/Project.toml`
 [no changes]
  Updating `/tmp/jl_BWJpcq/Manifest.toml`
 [no changes]
  Building LibPQ ────→ `~/.julia/packages/LibPQ/PeS3p/deps/build.log`
  Updating `/tmp/jl_Mt5fOV/Project.toml`
 [no changes]
  Updating `/tmp/jl_Mt5fOV/Manifest.toml`
 [no changes]
   Testing LibPQ
 Installed PooledArrays ─────── v0.5.3
 Installed OrderedCollections ─ v1.1.0
 Installed Missings ─────────── v0.4.3
 Installed SortingAlgorithms ── v0.3.1
 Installed Compat ───────────── v2.2.0
 Installed CategoricalArrays ── v0.7.7
 Installed Reexport ─────────── v0.2.0
 Installed InvertedIndices ──── v1.0.0
 Installed DataFrames ───────── v0.19.4
 Installed DataStructures ───── v0.17.9
  Updating `/tmp/jl_TnqV7d/Project.toml`
  [a93c6f00] + DataFrames v0.19.4
  Updating `/tmp/jl_TnqV7d/Manifest.toml`
  [324d7699] + CategoricalArrays v0.7.7
  [34da2185] + Compat v2.2.0
  [a93c6f00] + DataFrames v0.19.4
  [864edb3b] + DataStructures v0.17.9
  [41ab1584] + InvertedIndices v1.0.0
  [e1d29d7a] + Missings v0.4.3
  [bac558e1] + OrderedCollections v1.1.0
  [2dfb63ee] + PooledArrays v0.5.3
  [189a3867] + Reexport v0.2.0
  [a2af1166] + SortingAlgorithms v0.3.1
  [8bb1440f] + DelimitedFiles 
  [9fa8497b] + Future 
  [1a1011a3] + SharedArrays 
  [2f01184e] + SparseArrays 
  [10745b16] + Statistics 
Running sandbox
Status `/tmp/jl_TnqV7d/Project.toml`
  [b99e7846] BinaryProvider v0.5.8
  [fa961155] CEnum v0.2.0
  [a93c6f00] DataFrames v0.19.4
  [abce61dc] Decimals v0.4.0
  [ffbed154] DocStringExtensions v0.8.1
  [c8e1da08] IterTools v1.3.0
  [6f188dcb] LayerDicts v1.0.0
  [194296ae] LibPQ v1.0.6
  [f28f55f0] Memento v0.12.1
  [6fe1bfb0] OffsetArrays v0.11.4
  [bd369af6] Tables v0.2.11
  [f269a46b] TimeZones v0.10.5
  [ade2ca70] Dates 
  [7b1f6079] FileWatching 
  [8f399da3] Libdl 
  [8dfed614] Test 
Example SELECT: Test Failed at /home/pkgeval/.julia/packages/LibPQ/PeS3p/test/runtests.jl:76
  Expression: status(conn) == LibPQ.libpq_c.CONNECTION_OK
   Evaluated: LibPQ.libpq_c.CONNECTION_BAD == LibPQ.libpq_c.CONNECTION_OK
Stacktrace:
 [1] top-level scope at /home/pkgeval/.julia/packages/LibPQ/PeS3p/test/runtests.jl:76
 [2] top-level scope at /buildworker/worker/package_linux64/build/usr/share/julia/stdlib/v1.5/Test/src/Test.jl:1113
 [3] top-level scope at /home/pkgeval/.julia/packages/LibPQ/PeS3p/test/runtests.jl:73
 [4] top-level scope at /buildworker/worker/package_linux64/build/usr/share/julia/stdlib/v1.5/Test/src/Test.jl:1113
 [5] top-level scope at /home/pkgeval/.julia/packages/LibPQ/PeS3p/test/runtests.jl:70
 [6] top-level scope at /buildworker/worker/package_linux64/build/usr/share/julia/stdlib/v1.5/Test/src/Test.jl:1113
 [7] top-level scope at /home/pkgeval/.julia/packages/LibPQ/PeS3p/test/runtests.jl:34
Example SELECT: Test Failed at /home/pkgeval/.julia/packages/LibPQ/PeS3p/test/runtests.jl:89
  Expression: status(result) == LibPQ.libpq_c.PGRES_TUPLES_OK
   Evaluated: LibPQ.libpq_c.PGRES_FATAL_ERROR == LibPQ.libpq_c.PGRES_TUPLES_OK
Stacktrace:
 [1] top-level scope at /home/pkgeval/.julia/packages/LibPQ/PeS3p/test/runtests.jl:89
 [2] top-level scope at /buildworker/worker/package_linux64/build/usr/share/julia/stdlib/v1.5/Test/src/Test.jl:1113
 [3] top-level scope at /home/pkgeval/.julia/packages/LibPQ/PeS3p/test/runtests.jl:73
 [4] top-level scope at /buildworker/worker/package_linux64/build/usr/share/julia/stdlib/v1.5/Test/src/Test.jl:1113
 [5] top-level scope at /home/pkgeval/.julia/packages/LibPQ/PeS3p/test/runtests.jl:70
 [6] top-level scope at /buildworker/worker/package_linux64/build/usr/share/julia/stdlib/v1.5/Test/src/Test.jl:1113
 [7] top-level scope at /home/pkgeval/.julia/packages/LibPQ/PeS3p/test/runtests.jl:34
Example SELECT: Test Failed at /home/pkgeval/.julia/packages/LibPQ/PeS3p/test/runtests.jl:90
  Expression: isopen(result)
Stacktrace:
 [1] top-level scope at /home/pkgeval/.julia/packages/LibPQ/PeS3p/test/runtests.jl:90
 [2] top-level scope at /buildworker/worker/package_linux64/build/usr/share/julia/stdlib/v1.5/Test/src/Test.jl:1113
 [3] top-level scope at /home/pkgeval/.julia/packages/LibPQ/PeS3p/test/runtests.jl:73
 [4] top-level scope at /buildworker/worker/package_linux64/build/usr/share/julia/stdlib/v1.5/Test/src/Test.jl:1113
 [5] top-level scope at /home/pkgeval/.julia/packages/LibPQ/PeS3p/test/runtests.jl:70
 [6] top-level scope at /buildworker/worker/package_linux64/build/usr/share/julia/stdlib/v1.5/Test/src/Test.jl:1113
 [7] top-level scope at /home/pkgeval/.julia/packages/LibPQ/PeS3p/test/runtests.jl:34
Example SELECT: Test Failed at /home/pkgeval/.julia/packages/LibPQ/PeS3p/test/runtests.jl:91
  Expression: LibPQ.num_columns(result) == 1
   Evaluated: 0 == 1
Stacktrace:
 [1] top-level scope at /home/pkgeval/.julia/packages/LibPQ/PeS3p/test/runtests.jl:91
 [2] top-level scope at /buildworker/worker/package_linux64/build/usr/share/julia/stdlib/v1.5/Test/src/Test.jl:1113
 [3] top-level scope at /home/pkgeval/.julia/packages/LibPQ/PeS3p/test/runtests.jl:73
 [4] top-level scope at /buildworker/worker/package_linux64/build/usr/share/julia/stdlib/v1.5/Test/src/Test.jl:1113
 [5] top-level scope at /home/pkgeval/.julia/packages/LibPQ/PeS3p/test/runtests.jl:70
 [6] top-level scope at /buildworker/worker/package_linux64/build/usr/share/julia/stdlib/v1.5/Test/src/Test.jl:1113
 [7] top-level scope at /home/pkgeval/.julia/packages/LibPQ/PeS3p/test/runtests.jl:34
Example SELECT: Test Failed at /home/pkgeval/.julia/packages/LibPQ/PeS3p/test/runtests.jl:92
  Expression: LibPQ.num_rows(result) == 1
   Evaluated: 0 == 1
Stacktrace:
 [1] top-level scope at /home/pkgeval/.julia/packages/LibPQ/PeS3p/test/runtests.jl:92
 [2] top-level scope at /buildworker/worker/package_linux64/build/usr/share/julia/stdlib/v1.5/Test/src/Test.jl:1113
 [3] top-level scope at /home/pkgeval/.julia/packages/LibPQ/PeS3p/test/runtests.jl:73
 [4] top-level scope at /buildworker/worker/package_linux64/build/usr/share/julia/stdlib/v1.5/Test/src/Test.jl:1113
 [5] top-level scope at /home/pkgeval/.julia/packages/LibPQ/PeS3p/test/runtests.jl:70
 [6] top-level scope at /buildworker/worker/package_linux64/build/usr/share/julia/stdlib/v1.5/Test/src/Test.jl:1113
 [7] top-level scope at /home/pkgeval/.julia/packages/LibPQ/PeS3p/test/runtests.jl:34
Example SELECT: Error During Test at /home/pkgeval/.julia/packages/LibPQ/PeS3p/test/runtests.jl:93
  Test threw exception
  Expression: LibPQ.column_name(result, 1) == "typname"
  BoundsError: attempt to access 0-element Array{String,1} at index [1]
  Stacktrace:
   [1] getindex at ./array.jl:787 [inlined]
   [2] column_name(::LibPQ.Result, ::Int64) at /home/pkgeval/.julia/packages/LibPQ/PeS3p/src/results.jl:433
   [3] top-level scope at /home/pkgeval/.julia/packages/LibPQ/PeS3p/test/runtests.jl:93
   [4] top-level scope at /buildworker/worker/package_linux64/build/usr/share/julia/stdlib/v1.5/Test/src/Test.jl:1113
   [5] top-level scope at /home/pkgeval/.julia/packages/LibPQ/PeS3p/test/runtests.jl:73
   [6] top-level scope at /buildworker/worker/package_linux64/build/usr/share/julia/stdlib/v1.5/Test/src/Test.jl:1113
   [7] top-level scope at /home/pkgeval/.julia/packages/LibPQ/PeS3p/test/runtests.jl:70
   [8] top-level scope at /buildworker/worker/package_linux64/build/usr/share/julia/stdlib/v1.5/Test/src/Test.jl:1113
   [9] top-level scope at /home/pkgeval/.julia/packages/LibPQ/PeS3p/test/runtests.jl:34
  
Example SELECT: Test Failed at /home/pkgeval/.julia/packages/LibPQ/PeS3p/test/runtests.jl:94
  Expression: LibPQ.column_number(result, "typname") == 1
   Evaluated: 0 == 1
Stacktrace:
 [1] top-level scope at /home/pkgeval/.julia/packages/LibPQ/PeS3p/test/runtests.jl:94
 [2] top-level scope at /buildworker/worker/package_linux64/build/usr/share/julia/stdlib/v1.5/Test/src/Test.jl:1113
 [3] top-level scope at /home/pkgeval/.julia/packages/LibPQ/PeS3p/test/runtests.jl:73
 [4] top-level scope at /buildworker/worker/package_linux64/build/usr/share/julia/stdlib/v1.5/Test/src/Test.jl:1113
 [5] top-level scope at /home/pkgeval/.julia/packages/LibPQ/PeS3p/test/runtests.jl:70
 [6] top-level scope at /buildworker/worker/package_linux64/build/usr/share/julia/stdlib/v1.5/Test/src/Test.jl:1113
 [7] top-level scope at /home/pkgeval/.julia/packages/LibPQ/PeS3p/test/runtests.jl:34
Example SELECT: Error During Test at /home/pkgeval/.julia/packages/LibPQ/PeS3p/test/runtests.jl:98
  Test threw exception
  Expression: (data[:typname])[1] == "bool"
  type NamedTuple has no field typname
  Stacktrace:
   [1] getindex(::NamedTuple{(),Tuple{}}, ::Symbol) at ./namedtuple.jl:95
   [2] top-level scope at /home/pkgeval/.julia/packages/LibPQ/PeS3p/test/runtests.jl:98
   [3] top-level scope at /buildworker/worker/package_linux64/build/usr/share/julia/stdlib/v1.5/Test/src/Test.jl:1113
   [4] top-level scope at /home/pkgeval/.julia/packages/LibPQ/PeS3p/test/runtests.jl:73
   [5] top-level scope at /buildworker/worker/package_linux64/build/usr/share/julia/stdlib/v1.5/Test/src/Test.jl:1113
   [6] top-level scope at /home/pkgeval/.julia/packages/LibPQ/PeS3p/test/runtests.jl:70
   [7] top-level scope at /buildworker/worker/package_linux64/build/usr/share/julia/stdlib/v1.5/Test/src/Test.jl:1113
   [8] top-level scope at /home/pkgeval/.julia/packages/LibPQ/PeS3p/test/runtests.jl:34
  
Example SELECT: Test Failed at /home/pkgeval/.julia/packages/LibPQ/PeS3p/test/runtests.jl:111
  Expression: status(result) == LibPQ.libpq_c.PGRES_TUPLES_OK
   Evaluated: LibPQ.libpq_c.PGRES_FATAL_ERROR == LibPQ.libpq_c.PGRES_TUPLES_OK
Stacktrace:
 [1] top-level scope at /home/pkgeval/.julia/packages/LibPQ/PeS3p/test/runtests.jl:111
 [2] top-level scope at /buildworker/worker/package_linux64/build/usr/share/julia/stdlib/v1.5/Test/src/Test.jl:1113
 [3] top-level scope at /home/pkgeval/.julia/packages/LibPQ/PeS3p/test/runtests.jl:73
 [4] top-level scope at /buildworker/worker/package_linux64/build/usr/share/julia/stdlib/v1.5/Test/src/Test.jl:1113
 [5] top-level scope at /home/pkgeval/.julia/packages/LibPQ/PeS3p/test/runtests.jl:70
 [6] top-level scope at /buildworker/worker/package_linux64/build/usr/share/julia/stdlib/v1.5/Test/src/Test.jl:1113
 [7] top-level scope at /home/pkgeval/.julia/packages/LibPQ/PeS3p/test/runtests.jl:34
Example SELECT: Test Failed at /home/pkgeval/.julia/packages/LibPQ/PeS3p/test/runtests.jl:112
  Expression: isopen(result)
Stacktrace:
 [1] top-level scope at /home/pkgeval/.julia/packages/LibPQ/PeS3p/test/runtests.jl:112
 [2] top-level scope at /buildworker/worker/package_linux64/build/usr/share/julia/stdlib/v1.5/Test/src/Test.jl:1113
 [3] top-level scope at /home/pkgeval/.julia/packages/LibPQ/PeS3p/test/runtests.jl:73
 [4] top-level scope at /buildworker/worker/package_linux64/build/usr/share/julia/stdlib/v1.5/Test/src/Test.jl:1113
 [5] top-level scope at /home/pkgeval/.julia/packages/LibPQ/PeS3p/test/runtests.jl:70
 [6] top-level scope at /buildworker/worker/package_linux64/build/usr/share/julia/stdlib/v1.5/Test/src/Test.jl:1113
 [7] top-level scope at /home/pkgeval/.julia/packages/LibPQ/PeS3p/test/runtests.jl:34
Example SELECT: Test Failed at /home/pkgeval/.julia/packages/LibPQ/PeS3p/test/runtests.jl:113
  Expression: LibPQ.num_columns(result) == 1
   Evaluated: 0 == 1
Stacktrace:
 [1] top-level scope at /home/pkgeval/.julia/packages/LibPQ/PeS3p/test/runtests.jl:113
 [2] top-level scope at /buildworker/worker/package_linux64/build/usr/share/julia/stdlib/v1.5/Test/src/Test.jl:1113
 [3] top-level scope at /home/pkgeval/.julia/packages/LibPQ/PeS3p/test/runtests.jl:73
 [4] top-level scope at /buildworker/worker/package_linux64/build/usr/share/julia/stdlib/v1.5/Test/src/Test.jl:1113
 [5] top-level scope at /home/pkgeval/.julia/packages/LibPQ/PeS3p/test/runtests.jl:70
 [6] top-level scope at /buildworker/worker/package_linux64/build/usr/share/julia/stdlib/v1.5/Test/src/Test.jl:1113
 [7] top-level scope at /home/pkgeval/.julia/packages/LibPQ/PeS3p/test/runtests.jl:34
Example SELECT: Test Failed at /home/pkgeval/.julia/packages/LibPQ/PeS3p/test/runtests.jl:114
  Expression: LibPQ.num_rows(result) == 1
   Evaluated: 0 == 1
Stacktrace:
 [1] top-level scope at /home/pkgeval/.julia/packages/LibPQ/PeS3p/test/runtests.jl:114
 [2] top-level scope at /buildworker/worker/package_linux64/build/usr/share/julia/stdlib/v1.5/Test/src/Test.jl:1113
 [3] top-level scope at /home/pkgeval/.julia/packages/LibPQ/PeS3p/test/runtests.jl:73
 [4] top-level scope at /buildworker/worker/package_linux64/build/usr/share/julia/stdlib/v1.5/Test/src/Test.jl:1113
 [5] top-level scope at /home/pkgeval/.julia/packages/LibPQ/PeS3p/test/runtests.jl:70
 [6] top-level scope at /buildworker/worker/package_linux64/build/usr/share/julia/stdlib/v1.5/Test/src/Test.jl:1113
 [7] top-level scope at /home/pkgeval/.julia/packages/LibPQ/PeS3p/test/runtests.jl:34
Example SELECT: Error During Test at /home/pkgeval/.julia/packages/LibPQ/PeS3p/test/runtests.jl:115
  Test threw exception
  Expression: LibPQ.column_name(result, 1) == "typname"
  BoundsError: attempt to access 0-element Array{String,1} at index [1]
  Stacktrace:
   [1] getindex at ./array.jl:787 [inlined]
   [2] column_name(::LibPQ.Result, ::Int64) at /home/pkgeval/.julia/packages/LibPQ/PeS3p/src/results.jl:433
   [3] top-level scope at /home/pkgeval/.julia/packages/LibPQ/PeS3p/test/runtests.jl:115
   [4] top-level scope at /buildworker/worker/package_linux64/build/usr/share/julia/stdlib/v1.5/Test/src/Test.jl:1113
   [5] top-level scope at /home/pkgeval/.julia/packages/LibPQ/PeS3p/test/runtests.jl:73
   [6] top-level scope at /buildworker/worker/package_linux64/build/usr/share/julia/stdlib/v1.5/Test/src/Test.jl:1113
   [7] top-level scope at /home/pkgeval/.julia/packages/LibPQ/PeS3p/test/runtests.jl:70
   [8] top-level scope at /buildworker/worker/package_linux64/build/usr/share/julia/stdlib/v1.5/Test/src/Test.jl:1113
   [9] top-level scope at /home/pkgeval/.julia/packages/LibPQ/PeS3p/test/runtests.jl:34
  
Example SELECT: Error During Test at /home/pkgeval/.julia/packages/LibPQ/PeS3p/test/runtests.jl:119
  Test threw exception
  Expression: (data[:typname])[1] == "bool"
  type NamedTuple has no field typname
  Stacktrace:
   [1] getindex(::NamedTuple{(),Tuple{}}, ::Symbol) at ./namedtuple.jl:95
   [2] top-level scope at /home/pkgeval/.julia/packages/LibPQ/PeS3p/test/runtests.jl:119
   [3] top-level scope at /buildworker/worker/package_linux64/build/usr/share/julia/stdlib/v1.5/Test/src/Test.jl:1113
   [4] top-level scope at /home/pkgeval/.julia/packages/LibPQ/PeS3p/test/runtests.jl:73
   [5] top-level scope at /buildworker/worker/package_linux64/build/usr/share/julia/stdlib/v1.5/Test/src/Test.jl:1113
   [6] top-level scope at /home/pkgeval/.julia/packages/LibPQ/PeS3p/test/runtests.jl:70
   [7] top-level scope at /buildworker/worker/package_linux64/build/usr/share/julia/stdlib/v1.5/Test/src/Test.jl:1113
   [8] top-level scope at /home/pkgeval/.julia/packages/LibPQ/PeS3p/test/runtests.jl:34
  
Example SELECT: Error During Test at /home/pkgeval/.julia/packages/LibPQ/PeS3p/test/runtests.jl:72
  Got exception outside of a @test
  UnknownError: 
  Stacktrace:
   [1] error(::Logger, ::LibPQ.Errors.PQResultError{LibPQ.Errors.CUN,LibPQ.Errors.EUNOWN}) at /home/pkgeval/.julia/packages/Memento/UgJr2/src/loggers.jl:439
   [2] handle_result(::LibPQ.Result; throw_error::Bool) at /home/pkgeval/.julia/packages/LibPQ/PeS3p/src/results.jl:226
   [3] prepare(::LibPQ.Connection, ::String) at /home/pkgeval/.julia/packages/LibPQ/PeS3p/src/statements.jl:48
   [4] top-level scope at /home/pkgeval/.julia/packages/LibPQ/PeS3p/test/runtests.jl:127
   [5] top-level scope at /buildworker/worker/package_linux64/build/usr/share/julia/stdlib/v1.5/Test/src/Test.jl:1113
   [6] top-level scope at /home/pkgeval/.julia/packages/LibPQ/PeS3p/test/runtests.jl:73
   [7] top-level scope at /buildworker/worker/package_linux64/build/usr/share/julia/stdlib/v1.5/Test/src/Test.jl:1113
   [8] top-level scope at /home/pkgeval/.julia/packages/LibPQ/PeS3p/test/runtests.jl:70
   [9] top-level scope at /buildworker/worker/package_linux64/build/usr/share/julia/stdlib/v1.5/Test/src/Test.jl:1113
   [10] top-level scope at /home/pkgeval/.julia/packages/LibPQ/PeS3p/test/runtests.jl:34
   [11] include(::String) at ./client.jl:439
   [12] top-level scope at none:6
   [13] eval(::Module, ::Any) at ./boot.jl:331
   [14] exec_options(::Base.JLOptions) at ./client.jl:264
   [15] _start() at ./client.jl:484
  
Example INSERT and DELETE: Error During Test at /home/pkgeval/.julia/packages/LibPQ/PeS3p/test/runtests.jl:180
  Got exception outside of a @test
  could not connect to server: No such file or directory
  	Is the server running locally and accepting
  	connections on Unix domain socket "/tmp/.s.PGSQL.5432"?
  Stacktrace:
   [1] error(::Logger, ::LibPQ.Errors.PQConnectionError) at /home/pkgeval/.julia/packages/Memento/UgJr2/src/loggers.jl:439
   [2] handle_new_connection(::LibPQ.Connection; throw_error::Bool) at /home/pkgeval/.julia/packages/LibPQ/PeS3p/src/connections.jl:127
   [3] LibPQ.Connection(::String; throw_error::Bool, connect_timeout::Int64, options::Dict{String,String}, kwargs::Base.Iterators.Pairs{Union{},Union{},Tuple{},NamedTuple{(),Tuple{}}}) at /home/pkgeval/.julia/packages/LibPQ/PeS3p/src/connections.jl:320
   [4] LibPQ.Connection(::String) at /home/pkgeval/.julia/packages/LibPQ/PeS3p/src/connections.jl:272
   [5] top-level scope at /home/pkgeval/.julia/packages/LibPQ/PeS3p/test/runtests.jl:181
   [6] top-level scope at /buildworker/worker/package_linux64/build/usr/share/julia/stdlib/v1.5/Test/src/Test.jl:1113
   [7] top-level scope at /home/pkgeval/.julia/packages/LibPQ/PeS3p/test/runtests.jl:181
   [8] top-level scope at /buildworker/worker/package_linux64/build/usr/share/julia/stdlib/v1.5/Test/src/Test.jl:1113
   [9] top-level scope at /home/pkgeval/.julia/packages/LibPQ/PeS3p/test/runtests.jl:70
   [10] top-level scope at /buildworker/worker/package_linux64/build/usr/share/julia/stdlib/v1.5/Test/src/Test.jl:1113
   [11] top-level scope at /home/pkgeval/.julia/packages/LibPQ/PeS3p/test/runtests.jl:34
   [12] include(::String) at ./client.jl:439
   [13] top-level scope at none:6
   [14] eval(::Module, ::Any) at ./boot.jl:331
   [15] exec_options(::Base.JLOptions) at ./client.jl:264
   [16] _start() at ./client.jl:484
  
Example COPY FROM: Error During Test at /home/pkgeval/.julia/packages/LibPQ/PeS3p/test/runtests.jl:277
  Got exception outside of a @test
  could not connect to server: No such file or directory
  	Is the server running locally and accepting
  	connections on Unix domain socket "/tmp/.s.PGSQL.5432"?
  Stacktrace:
   [1] error(::Logger, ::LibPQ.Errors.PQConnectionError) at /home/pkgeval/.julia/packages/Memento/UgJr2/src/loggers.jl:439
   [2] handle_new_connection(::LibPQ.Connection; throw_error::Bool) at /home/pkgeval/.julia/packages/LibPQ/PeS3p/src/connections.jl:127
   [3] LibPQ.Connection(::String; throw_error::Bool, connect_timeout::Int64, options::Dict{String,String}, kwargs::Base.Iterators.Pairs{Union{},Union{},Tuple{},NamedTuple{(),Tuple{}}}) at /home/pkgeval/.julia/packages/LibPQ/PeS3p/src/connections.jl:320
   [4] LibPQ.Connection(::String) at /home/pkgeval/.julia/packages/LibPQ/PeS3p/src/connections.jl:272
   [5] top-level scope at /home/pkgeval/.julia/packages/LibPQ/PeS3p/test/runtests.jl:278
   [6] top-level scope at /buildworker/worker/package_linux64/build/usr/share/julia/stdlib/v1.5/Test/src/Test.jl:1113
   [7] top-level scope at /home/pkgeval/.julia/packages/LibPQ/PeS3p/test/runtests.jl:278
   [8] top-level scope at /buildworker/worker/package_linux64/build/usr/share/julia/stdlib/v1.5/Test/src/Test.jl:1113
   [9] top-level scope at /home/pkgeval/.julia/packages/LibPQ/PeS3p/test/runtests.jl:277
   [10] top-level scope at /buildworker/worker/package_linux64/build/usr/share/julia/stdlib/v1.5/Test/src/Test.jl:1113
   [11] top-level scope at /home/pkgeval/.julia/packages/LibPQ/PeS3p/test/runtests.jl:70
   [12] top-level scope at /buildworker/worker/package_linux64/build/usr/share/julia/stdlib/v1.5/Test/src/Test.jl:1113
   [13] top-level scope at /home/pkgeval/.julia/packages/LibPQ/PeS3p/test/runtests.jl:34
   [14] include(::String) at ./client.jl:439
   [15] top-level scope at none:6
   [16] eval(::Module, ::Any) at ./boot.jl:331
   [17] exec_options(::Base.JLOptions) at ./client.jl:264
   [18] _start() at ./client.jl:484
  
Wrong column order: Error During Test at /home/pkgeval/.julia/packages/LibPQ/PeS3p/test/runtests.jl:321
  Got exception outside of a @test
  could not connect to server: No such file or directory
  	Is the server running locally and accepting
  	connections on Unix domain socket "/tmp/.s.PGSQL.5432"?
  Stacktrace:
   [1] error(::Logger, ::LibPQ.Errors.PQConnectionError) at /home/pkgeval/.julia/packages/Memento/UgJr2/src/loggers.jl:439
   [2] handle_new_connection(::LibPQ.Connection; throw_error::Bool) at /home/pkgeval/.julia/packages/LibPQ/PeS3p/src/connections.jl:127
   [3] LibPQ.Connection(::String; throw_error::Bool, connect_timeout::Int64, options::Dict{String,String}, kwargs::Base.Iterators.Pairs{Union{},Union{},Tuple{},NamedTuple{(),Tuple{}}}) at /home/pkgeval/.julia/packages/LibPQ/PeS3p/src/connections.jl:320
   [4] LibPQ.Connection(::String) at /home/pkgeval/.julia/packages/LibPQ/PeS3p/src/connections.jl:272
   [5] top-level scope at /home/pkgeval/.julia/packages/LibPQ/PeS3p/test/runtests.jl:322
   [6] top-level scope at /buildworker/worker/package_linux64/build/usr/share/julia/stdlib/v1.5/Test/src/Test.jl:1113
   [7] top-level scope at /home/pkgeval/.julia/packages/LibPQ/PeS3p/test/runtests.jl:322
   [8] top-level scope at /buildworker/worker/package_linux64/build/usr/share/julia/stdlib/v1.5/Test/src/Test.jl:1113
   [9] top-level scope at /home/pkgeval/.julia/packages/LibPQ/PeS3p/test/runtests.jl:277
   [10] top-level scope at /buildworker/worker/package_linux64/build/usr/share/julia/stdlib/v1.5/Test/src/Test.jl:1113
   [11] top-level scope at /home/pkgeval/.julia/packages/LibPQ/PeS3p/test/runtests.jl:70
   [12] top-level scope at /buildworker/worker/package_linux64/build/usr/share/julia/stdlib/v1.5/Test/src/Test.jl:1113
   [13] top-level scope at /home/pkgeval/.julia/packages/LibPQ/PeS3p/test/runtests.jl:34
   [14] include(::String) at ./client.jl:439
   [15] top-level scope at none:6
   [16] eval(::Module, ::Any) at ./boot.jl:331
   [17] exec_options(::Base.JLOptions) at ./client.jl:264
   [18] _start() at ./client.jl:484
  
do: Error During Test at /home/pkgeval/.julia/packages/LibPQ/PeS3p/test/runtests.jl:376
  Got exception outside of a @test
  could not connect to server: No such file or directory
  	Is the server running locally and accepting
  	connections on Unix domain socket "/tmp/.s.PGSQL.5432"?
  Stacktrace:
   [1] error(::Logger, ::LibPQ.Errors.PQConnectionError) at /home/pkgeval/.julia/packages/Memento/UgJr2/src/loggers.jl:439
   [2] handle_new_connection(::LibPQ.Connection; throw_error::Bool) at /home/pkgeval/.julia/packages/LibPQ/PeS3p/src/connections.jl:127
   [3] LibPQ.Connection(::String; throw_error::Bool, connect_timeout::Int64, options::Dict{String,String}, kwargs::Base.Iterators.Pairs{Union{},Union{},Tuple{},NamedTuple{(),Tuple{}}}) at /home/pkgeval/.julia/packages/LibPQ/PeS3p/src/connections.jl:320
   [4] LibPQ.Connection(::var"#6#35", ::String; kwargs::Base.Iterators.Pairs{Symbol,Bool,Tuple{Symbol},NamedTuple{(:throw_error,),Tuple{Bool}}}) at /home/pkgeval/.julia/packages/LibPQ/PeS3p/src/connections.jl:350
   [5] top-level scope at /home/pkgeval/.julia/packages/LibPQ/PeS3p/test/runtests.jl:379
   [6] top-level scope at /buildworker/worker/package_linux64/build/usr/share/julia/stdlib/v1.5/Test/src/Test.jl:1113
   [7] top-level scope at /home/pkgeval/.julia/packages/LibPQ/PeS3p/test/runtests.jl:377
   [8] top-level scope at /buildworker/worker/package_linux64/build/usr/share/julia/stdlib/v1.5/Test/src/Test.jl:1113
   [9] top-level scope at /home/pkgeval/.julia/packages/LibPQ/PeS3p/test/runtests.jl:376
   [10] top-level scope at /buildworker/worker/package_linux64/build/usr/share/julia/stdlib/v1.5/Test/src/Test.jl:1113
   [11] top-level scope at /home/pkgeval/.julia/packages/LibPQ/PeS3p/test/runtests.jl:70
   [12] top-level scope at /buildworker/worker/package_linux64/build/usr/share/julia/stdlib/v1.5/Test/src/Test.jl:1113
   [13] top-level scope at /home/pkgeval/.julia/packages/LibPQ/PeS3p/test/runtests.jl:34
   [14] include(::String) at ./client.jl:439
   [15] top-level scope at none:6
   [16] eval(::Module, ::Any) at ./boot.jl:331
   [17] exec_options(::Base.JLOptions) at ./client.jl:264
   [18] _start() at ./client.jl:484
  
Version Numbers: Error During Test at /home/pkgeval/.julia/packages/LibPQ/PeS3p/test/runtests.jl:395
  Got exception outside of a @test
  could not connect to server: No such file or directory
  	Is the server running locally and accepting
  	connections on Unix domain socket "/tmp/.s.PGSQL.5432"?
  Stacktrace:
   [1] error(::Logger, ::LibPQ.Errors.PQConnectionError) at /home/pkgeval/.julia/packages/Memento/UgJr2/src/loggers.jl:439
   [2] handle_new_connection(::LibPQ.Connection; throw_error::Bool) at /home/pkgeval/.julia/packages/LibPQ/PeS3p/src/connections.jl:127
   [3] LibPQ.Connection(::String; throw_error::Bool, connect_timeout::Int64, options::Dict{String,String}, kwargs::Base.Iterators.Pairs{Union{},Union{},Tuple{},NamedTuple{(),Tuple{}}}) at /home/pkgeval/.julia/packages/LibPQ/PeS3p/src/connections.jl:320
   [4] top-level scope at /home/pkgeval/.julia/packages/LibPQ/PeS3p/test/runtests.jl:396
   [5] top-level scope at /buildworker/worker/package_linux64/build/usr/share/julia/stdlib/v1.5/Test/src/Test.jl:1113
   [6] top-level scope at /home/pkgeval/.julia/packages/LibPQ/PeS3p/test/runtests.jl:396
   [7] top-level scope at /buildworker/worker/package_linux64/build/usr/share/julia/stdlib/v1.5/Test/src/Test.jl:1113
   [8] top-level scope at /home/pkgeval/.julia/packages/LibPQ/PeS3p/test/runtests.jl:376
   [9] top-level scope at /buildworker/worker/package_linux64/build/usr/share/julia/stdlib/v1.5/Test/src/Test.jl:1113
   [10] top-level scope at /home/pkgeval/.julia/packages/LibPQ/PeS3p/test/runtests.jl:70
   [11] top-level scope at /buildworker/worker/package_linux64/build/usr/share/julia/stdlib/v1.5/Test/src/Test.jl:1113
   [12] top-level scope at /home/pkgeval/.julia/packages/LibPQ/PeS3p/test/runtests.jl:34
   [13] include(::String) at ./client.jl:439
   [14] top-level scope at none:6
   [15] eval(::Module, ::Any) at ./boot.jl:331
   [16] exec_options(::Base.JLOptions) at ./client.jl:264
   [17] _start() at ./client.jl:484
  
Encoding: Error During Test at /home/pkgeval/.julia/packages/LibPQ/PeS3p/test/runtests.jl:402
  Got exception outside of a @test
  could not connect to server: No such file or directory
  	Is the server running locally and accepting
  	connections on Unix domain socket "/tmp/.s.PGSQL.5432"?
  Stacktrace:
   [1] error(::Logger, ::LibPQ.Errors.PQConnectionError) at /home/pkgeval/.julia/packages/Memento/UgJr2/src/loggers.jl:439
   [2] handle_new_connection(::LibPQ.Connection; throw_error::Bool) at /home/pkgeval/.julia/packages/LibPQ/PeS3p/src/connections.jl:127
   [3] LibPQ.Connection(::String; throw_error::Bool, connect_timeout::Int64, options::Dict{String,String}, kwargs::Base.Iterators.Pairs{Union{},Union{},Tuple{},NamedTuple{(),Tuple{}}}) at /home/pkgeval/.julia/packages/LibPQ/PeS3p/src/connections.jl:320
   [4] top-level scope at /home/pkgeval/.julia/packages/LibPQ/PeS3p/test/runtests.jl:403
   [5] top-level scope at /buildworker/worker/package_linux64/build/usr/share/julia/stdlib/v1.5/Test/src/Test.jl:1113
   [6] top-level scope at /home/pkgeval/.julia/packages/LibPQ/PeS3p/test/runtests.jl:403
   [7] top-level scope at /buildworker/worker/package_linux64/build/usr/share/julia/stdlib/v1.5/Test/src/Test.jl:1113
   [8] top-level scope at /home/pkgeval/.julia/packages/LibPQ/PeS3p/test/runtests.jl:376
   [9] top-level scope at /buildworker/worker/package_linux64/build/usr/share/julia/stdlib/v1.5/Test/src/Test.jl:1113
   [10] top-level scope at /home/pkgeval/.julia/packages/LibPQ/PeS3p/test/runtests.jl:70
   [11] top-level scope at /buildworker/worker/package_linux64/build/usr/share/julia/stdlib/v1.5/Test/src/Test.jl:1113
   [12] top-level scope at /home/pkgeval/.julia/packages/LibPQ/PeS3p/test/runtests.jl:34
   [13] include(::String) at ./client.jl:439
   [14] top-level scope at none:6
   [15] eval(::Module, ::Any) at ./boot.jl:331
   [16] exec_options(::Base.JLOptions) at ./client.jl:264
   [17] _start() at ./client.jl:484
  
Options: Error During Test at /home/pkgeval/.julia/packages/LibPQ/PeS3p/test/runtests.jl:428
  Got exception outside of a @test
  could not connect to server: No such file or directory
  	Is the server running locally and accepting
  	connections on Unix domain socket "/tmp/.s.PGSQL.5432"?
  Stacktrace:
   [1] error(::Logger, ::LibPQ.Errors.PQConnectionError) at /home/pkgeval/.julia/packages/Memento/UgJr2/src/loggers.jl:439
   [2] handle_new_connection(::LibPQ.Connection; throw_error::Bool) at /home/pkgeval/.julia/packages/LibPQ/PeS3p/src/connections.jl:127
   [3] LibPQ.Connection(::String; throw_error::Bool, connect_timeout::Int64, options::Dict{String,String}, kwargs::Base.Iterators.Pairs{Symbol,Dict{Symbol,DataType},Tuple{Symbol},NamedTuple{(:type_map,),Tuple{Dict{Symbol,DataType}}}}) at /home/pkgeval/.julia/packages/LibPQ/PeS3p/src/connections.jl:320
   [4] top-level scope at /home/pkgeval/.julia/packages/LibPQ/PeS3p/test/runtests.jl:429
   [5] top-level scope at /buildworker/worker/package_linux64/build/usr/share/julia/stdlib/v1.5/Test/src/Test.jl:1113
   [6] top-level scope at /home/pkgeval/.julia/packages/LibPQ/PeS3p/test/runtests.jl:429
   [7] top-level scope at /buildworker/worker/package_linux64/build/usr/share/julia/stdlib/v1.5/Test/src/Test.jl:1113
   [8] top-level scope at /home/pkgeval/.julia/packages/LibPQ/PeS3p/test/runtests.jl:376
   [9] top-level scope at /buildworker/worker/package_linux64/build/usr/share/julia/stdlib/v1.5/Test/src/Test.jl:1113
   [10] top-level scope at /home/pkgeval/.julia/packages/LibPQ/PeS3p/test/runtests.jl:70
   [11] top-level scope at /buildworker/worker/package_linux64/build/usr/share/julia/stdlib/v1.5/Test/src/Test.jl:1113
   [12] top-level scope at /home/pkgeval/.julia/packages/LibPQ/PeS3p/test/runtests.jl:34
   [13] include(::String) at ./client.jl:439
   [14] top-level scope at none:6
   [15] eval(::Module, ::Any) at ./boot.jl:331
   [16] exec_options(::Base.JLOptions) at ./client.jl:264
   [17] _start() at ./client.jl:484
  
Time Zone: Error During Test at /home/pkgeval/.julia/packages/LibPQ/PeS3p/test/runtests.jl:447
  Got exception outside of a @test
  could not connect to server: No such file or directory
  	Is the server running locally and accepting
  	connections on Unix domain socket "/tmp/.s.PGSQL.5432"?
  Stacktrace:
   [1] error(::Logger, ::LibPQ.Errors.PQConnectionError) at /home/pkgeval/.julia/packages/Memento/UgJr2/src/loggers.jl:439
   [2] handle_new_connection(::LibPQ.Connection; throw_error::Bool) at /home/pkgeval/.julia/packages/LibPQ/PeS3p/src/connections.jl:127
   [3] LibPQ.Connection(::String; throw_error::Bool, connect_timeout::Int64, options::Dict{String,String}, kwargs::Base.Iterators.Pairs{Union{},Union{},Tuple{},NamedTuple{(),Tuple{}}}) at /home/pkgeval/.julia/packages/LibPQ/PeS3p/src/connections.jl:320
   [4] LibPQ.Connection(::var"#10#40"{var"#connection_tz#38"}, ::String; kwargs::Base.Iterators.Pairs{Symbol,Bool,Tuple{Symbol},NamedTuple{(:throw_error,),Tuple{Bool}}}) at /home/pkgeval/.julia/packages/LibPQ/PeS3p/src/connections.jl:350
   [5] (::var"#9#39"{String,var"#connection_tz#38"})() at /home/pkgeval/.julia/packages/LibPQ/PeS3p/test/runtests.jl:464
   [6] withenv(::var"#9#39"{String,var"#connection_tz#38"}, ::Pair{String,Nothing}) at ./env.jl:161
   [7] top-level scope at /home/pkgeval/.julia/packages/LibPQ/PeS3p/test/runtests.jl:463
   [8] top-level scope at /buildworker/worker/package_linux64/build/usr/share/julia/stdlib/v1.5/Test/src/Test.jl:1113
   [9] top-level scope at /home/pkgeval/.julia/packages/LibPQ/PeS3p/test/runtests.jl:448
   [10] top-level scope at /buildworker/worker/package_linux64/build/usr/share/julia/stdlib/v1.5/Test/src/Test.jl:1113
   [11] top-level scope at /home/pkgeval/.julia/packages/LibPQ/PeS3p/test/runtests.jl:376
   [12] top-level scope at /buildworker/worker/package_linux64/build/usr/share/julia/stdlib/v1.5/Test/src/Test.jl:1113
   [13] top-level scope at /home/pkgeval/.julia/packages/LibPQ/PeS3p/test/runtests.jl:70
   [14] top-level scope at /buildworker/worker/package_linux64/build/usr/share/julia/stdlib/v1.5/Test/src/Test.jl:1113
   [15] top-level scope at /home/pkgeval/.julia/packages/LibPQ/PeS3p/test/runtests.jl:34
   [16] include(::String) at ./client.jl:439
   [17] top-level scope at none:6
   [18] eval(::Module, ::Any) at ./boot.jl:331
   [19] exec_options(::Base.JLOptions) at ./client.jl:264
   [20] _start() at ./client.jl:484
  
Finalizer: Error During Test at /home/pkgeval/.julia/packages/LibPQ/PeS3p/test/runtests.jl:548
  Got exception outside of a @test
  could not connect to server: No such file or directory
  	Is the server running locally and accepting
  	connections on Unix domain socket "/tmp/.s.PGSQL.5432"?
  Stacktrace:
   [1] error(::Logger, ::LibPQ.Errors.PQConnectionError) at /home/pkgeval/.julia/packages/Memento/UgJr2/src/loggers.jl:439
   [2] handle_new_connection(::LibPQ.Connection; throw_error::Bool) at /home/pkgeval/.julia/packages/LibPQ/PeS3p/src/connections.jl:127
   [3] LibPQ.Connection(::String; throw_error::Bool, connect_timeout::Int64, options::Dict{String,String}, kwargs::Base.Iterators.Pairs{Union{},Union{},Tuple{},NamedTuple{(),Tuple{}}}) at /home/pkgeval/.julia/packages/LibPQ/PeS3p/src/connections.jl:320
   [4] Connection at /home/pkgeval/.julia/packages/LibPQ/PeS3p/src/connections.jl:272 [inlined]
   [5] #21 at /home/pkgeval/.julia/packages/LibPQ/PeS3p/test/runtests.jl:550 [inlined]
   [6] iterate at ./generator.jl:47 [inlined]
   [7] _collect(::UnitRange{Int64}, ::Base.Generator{UnitRange{Int64},var"#21#51"{String}}, ::Base.EltypeUnknown, ::Base.HasShape{1}) at ./array.jl:678
   [8] collect_similar(::UnitRange{Int64}, ::Base.Generator{UnitRange{Int64},var"#21#51"{String}}) at ./array.jl:607
   [9] map(::Function, ::UnitRange{Int64}) at ./abstractarray.jl:2072
   [10] top-level scope at /home/pkgeval/.julia/packages/LibPQ/PeS3p/test/runtests.jl:549
   [11] top-level scope at /buildworker/worker/package_linux64/build/usr/share/julia/stdlib/v1.5/Test/src/Test.jl:1113
   [12] top-level scope at /home/pkgeval/.julia/packages/LibPQ/PeS3p/test/runtests.jl:549
   [13] top-level scope at /buildworker/worker/package_linux64/build/usr/share/julia/stdlib/v1.5/Test/src/Test.jl:1113
   [14] top-level scope at /home/pkgeval/.julia/packages/LibPQ/PeS3p/test/runtests.jl:376
   [15] top-level scope at /buildworker/worker/package_linux64/build/usr/share/julia/stdlib/v1.5/Test/src/Test.jl:1113
   [16] top-level scope at /home/pkgeval/.julia/packages/LibPQ/PeS3p/test/runtests.jl:70
   [17] top-level scope at /buildworker/worker/package_linux64/build/usr/share/julia/stdlib/v1.5/Test/src/Test.jl:1113
   [18] top-level scope at /home/pkgeval/.julia/packages/LibPQ/PeS3p/test/runtests.jl:34
   [19] include(::String) at ./client.jl:439
   [20] top-level scope at none:6
   [21] eval(::Module, ::Any) at ./boot.jl:331
   [22] exec_options(::Base.JLOptions) at ./client.jl:264
   [23] _start() at ./client.jl:484
  
closed socket (EBADF): Error During Test at /home/pkgeval/.julia/packages/LibPQ/PeS3p/test/runtests.jl:609
  Got exception outside of a @test
  SystemError: fdopen: Bad file descriptor
  Stacktrace:
   [1] systemerror(::String, ::Int32; extrainfo::Nothing) at ./error.jl:168
   [2] #systemerror#50 at ./error.jl:167 [inlined]
   [3] systemerror at ./error.jl:167 [inlined]
   [4] Base.Libc.FILE(::RawFD, ::String) at ./libc.jl:82
   [5] top-level scope at /home/pkgeval/.julia/packages/LibPQ/PeS3p/test/runtests.jl:611
   [6] top-level scope at /buildworker/worker/package_linux64/build/usr/share/julia/stdlib/v1.5/Test/src/Test.jl:1113
   [7] top-level scope at /home/pkgeval/.julia/packages/LibPQ/PeS3p/test/runtests.jl:610
   [8] top-level scope at /buildworker/worker/package_linux64/build/usr/share/julia/stdlib/v1.5/Test/src/Test.jl:1113
   [9] top-level scope at /home/pkgeval/.julia/packages/LibPQ/PeS3p/test/runtests.jl:593
   [10] top-level scope at /buildworker/worker/package_linux64/build/usr/share/julia/stdlib/v1.5/Test/src/Test.jl:1113
   [11] top-level scope at /home/pkgeval/.julia/packages/LibPQ/PeS3p/test/runtests.jl:376
   [12] top-level scope at /buildworker/worker/package_linux64/build/usr/share/julia/stdlib/v1.5/Test/src/Test.jl:1113
   [13] top-level scope at /home/pkgeval/.julia/packages/LibPQ/PeS3p/test/runtests.jl:70
   [14] top-level scope at /buildworker/worker/package_linux64/build/usr/share/julia/stdlib/v1.5/Test/src/Test.jl:1113
   [15] top-level scope at /home/pkgeval/.julia/packages/LibPQ/PeS3p/test/runtests.jl:34
   [16] include(::String) at ./client.jl:439
   [17] top-level scope at none:6
   [18] eval(::Module, ::Any) at ./boot.jl:331
   [19] exec_options(::Base.JLOptions) at ./client.jl:264
   [20] _start() at ./client.jl:484
  
Nulls: Error During Test at /home/pkgeval/.julia/packages/LibPQ/PeS3p/test/runtests.jl:657
  Got exception outside of a @test
  could not connect to server: No such file or directory
  	Is the server running locally and accepting
  	connections on Unix domain socket "/tmp/.s.PGSQL.5432"?
  Stacktrace:
   [1] error(::Logger, ::LibPQ.Errors.PQConnectionError) at /home/pkgeval/.julia/packages/Memento/UgJr2/src/loggers.jl:439
   [2] handle_new_connection(::LibPQ.Connection; throw_error::Bool) at /home/pkgeval/.julia/packages/LibPQ/PeS3p/src/connections.jl:127
   [3] LibPQ.Connection(::String; throw_error::Bool, connect_timeout::Int64, options::Dict{String,String}, kwargs::Base.Iterators.Pairs{Union{},Union{},Tuple{},NamedTuple{(),Tuple{}}}) at /home/pkgeval/.julia/packages/LibPQ/PeS3p/src/connections.jl:320
   [4] top-level scope at /home/pkgeval/.julia/packages/LibPQ/PeS3p/test/runtests.jl:658
   [5] top-level scope at /buildworker/worker/package_linux64/build/usr/share/julia/stdlib/v1.5/Test/src/Test.jl:1113
   [6] top-level scope at /home/pkgeval/.julia/packages/LibPQ/PeS3p/test/runtests.jl:658
   [7] top-level scope at /buildworker/worker/package_linux64/build/usr/share/julia/stdlib/v1.5/Test/src/Test.jl:1113
   [8] top-level scope at /home/pkgeval/.julia/packages/LibPQ/PeS3p/test/runtests.jl:657
   [9] top-level scope at /buildworker/worker/package_linux64/build/usr/share/julia/stdlib/v1.5/Test/src/Test.jl:1113
   [10] top-level scope at /home/pkgeval/.julia/packages/LibPQ/PeS3p/test/runtests.jl:70
   [11] top-level scope at /buildworker/worker/package_linux64/build/usr/share/julia/stdlib/v1.5/Test/src/Test.jl:1113
   [12] top-level scope at /home/pkgeval/.julia/packages/LibPQ/PeS3p/test/runtests.jl:34
   [13] include(::String) at ./client.jl:439
   [14] top-level scope at none:6
   [15] eval(::Module, ::Any) at ./boot.jl:331
   [16] exec_options(::Base.JLOptions) at ./client.jl:264
   [17] _start() at ./client.jl:484
  
Not Nulls: Error During Test at /home/pkgeval/.julia/packages/LibPQ/PeS3p/test/runtests.jl:741
  Got exception outside of a @test
  could not connect to server: No such file or directory
  	Is the server running locally and accepting
  	connections on Unix domain socket "/tmp/.s.PGSQL.5432"?
  Stacktrace:
   [1] error(::Logger, ::LibPQ.Errors.PQConnectionError) at /home/pkgeval/.julia/packages/Memento/UgJr2/src/loggers.jl:439
   [2] handle_new_connection(::LibPQ.Connection; throw_error::Bool) at /home/pkgeval/.julia/packages/LibPQ/PeS3p/src/connections.jl:127
   [3] LibPQ.Connection(::String; throw_error::Bool, connect_timeout::Int64, options::Dict{String,String}, kwargs::Base.Iterators.Pairs{Union{},Union{},Tuple{},NamedTuple{(),Tuple{}}}) at /home/pkgeval/.julia/packages/LibPQ/PeS3p/src/connections.jl:320
   [4] top-level scope at /home/pkgeval/.julia/packages/LibPQ/PeS3p/test/runtests.jl:742
   [5] top-level scope at /buildworker/worker/package_linux64/build/usr/share/julia/stdlib/v1.5/Test/src/Test.jl:1113
   [6] top-level scope at /home/pkgeval/.julia/packages/LibPQ/PeS3p/test/runtests.jl:742
   [7] top-level scope at /buildworker/worker/package_linux64/build/usr/share/julia/stdlib/v1.5/Test/src/Test.jl:1113
   [8] top-level scope at /home/pkgeval/.julia/packages/LibPQ/PeS3p/test/runtests.jl:657
   [9] top-level scope at /buildworker/worker/package_linux64/build/usr/share/julia/stdlib/v1.5/Test/src/Test.jl:1113
   [10] top-level scope at /home/pkgeval/.julia/packages/LibPQ/PeS3p/test/runtests.jl:70
   [11] top-level scope at /buildworker/worker/package_linux64/build/usr/share/julia/stdlib/v1.5/Test/src/Test.jl:1113
   [12] top-level scope at /home/pkgeval/.julia/packages/LibPQ/PeS3p/test/runtests.jl:34
   [13] include(::String) at ./client.jl:439
   [14] top-level scope at none:6
   [15] eval(::Module, ::Any) at ./boot.jl:331
   [16] exec_options(::Base.JLOptions) at ./client.jl:264
   [17] _start() at ./client.jl:484
  
Tables.jl: Error During Test at /home/pkgeval/.julia/packages/LibPQ/PeS3p/test/runtests.jl:820
  Got exception outside of a @test
  could not connect to server: No such file or directory
  	Is the server running locally and accepting
  	connections on Unix domain socket "/tmp/.s.PGSQL.5432"?
  Stacktrace:
   [1] error(::Logger, ::LibPQ.Errors.PQConnectionError) at /home/pkgeval/.julia/packages/Memento/UgJr2/src/loggers.jl:439
   [2] handle_new_connection(::LibPQ.Connection; throw_error::Bool) at /home/pkgeval/.julia/packages/LibPQ/PeS3p/src/connections.jl:127
   [3] LibPQ.Connection(::String; throw_error::Bool, connect_timeout::Int64, options::Dict{String,String}, kwargs::Base.Iterators.Pairs{Union{},Union{},Tuple{},NamedTuple{(),Tuple{}}}) at /home/pkgeval/.julia/packages/LibPQ/PeS3p/src/connections.jl:320
   [4] top-level scope at /home/pkgeval/.julia/packages/LibPQ/PeS3p/test/runtests.jl:821
   [5] top-level scope at /buildworker/worker/package_linux64/build/usr/share/julia/stdlib/v1.5/Test/src/Test.jl:1113
   [6] top-level scope at /home/pkgeval/.julia/packages/LibPQ/PeS3p/test/runtests.jl:821
   [7] top-level scope at /buildworker/worker/package_linux64/build/usr/share/julia/stdlib/v1.5/Test/src/Test.jl:1113
   [8] top-level scope at /home/pkgeval/.julia/packages/LibPQ/PeS3p/test/runtests.jl:657
   [9] top-level scope at /buildworker/worker/package_linux64/build/usr/share/julia/stdlib/v1.5/Test/src/Test.jl:1113
   [10] top-level scope at /home/pkgeval/.julia/packages/LibPQ/PeS3p/test/runtests.jl:70
   [11] top-level scope at /buildworker/worker/package_linux64/build/usr/share/julia/stdlib/v1.5/Test/src/Test.jl:1113
   [12] top-level scope at /home/pkgeval/.julia/packages/LibPQ/PeS3p/test/runtests.jl:34
   [13] include(::String) at ./client.jl:439
   [14] top-level scope at none:6
   [15] eval(::Module, ::Any) at ./boot.jl:331
   [16] exec_options(::Base.JLOptions) at ./client.jl:264
   [17] _start() at ./client.jl:484
  
Duplicate names: Error During Test at /home/pkgeval/.julia/packages/LibPQ/PeS3p/test/runtests.jl:862
  Got exception outside of a @test
  could not connect to server: No such file or directory
  	Is the server running locally and accepting
  	connections on Unix domain socket "/tmp/.s.PGSQL.5432"?
  Stacktrace:
   [1] error(::Logger, ::LibPQ.Errors.PQConnectionError) at /home/pkgeval/.julia/packages/Memento/UgJr2/src/loggers.jl:439
   [2] handle_new_connection(::LibPQ.Connection; throw_error::Bool) at /home/pkgeval/.julia/packages/LibPQ/PeS3p/src/connections.jl:127
   [3] LibPQ.Connection(::String; throw_error::Bool, connect_timeout::Int64, options::Dict{String,String}, kwargs::Base.Iterators.Pairs{Union{},Union{},Tuple{},NamedTuple{(),Tuple{}}}) at /home/pkgeval/.julia/packages/LibPQ/PeS3p/src/connections.jl:320
   [4] top-level scope at /home/pkgeval/.julia/packages/LibPQ/PeS3p/test/runtests.jl:863
   [5] top-level scope at /buildworker/worker/package_linux64/build/usr/share/julia/stdlib/v1.5/Test/src/Test.jl:1113
   [6] top-level scope at /home/pkgeval/.julia/packages/LibPQ/PeS3p/test/runtests.jl:863
   [7] top-level scope at /buildworker/worker/package_linux64/build/usr/share/julia/stdlib/v1.5/Test/src/Test.jl:1113
   [8] top-level scope at /home/pkgeval/.julia/packages/LibPQ/PeS3p/test/runtests.jl:657
   [9] top-level scope at /buildworker/worker/package_linux64/build/usr/share/julia/stdlib/v1.5/Test/src/Test.jl:1113
   [10] top-level scope at /home/pkgeval/.julia/packages/LibPQ/PeS3p/test/runtests.jl:70
   [11] top-level scope at /buildworker/worker/package_linux64/build/usr/share/julia/stdlib/v1.5/Test/src/Test.jl:1113
   [12] top-level scope at /home/pkgeval/.julia/packages/LibPQ/PeS3p/test/runtests.jl:34
   [13] include(::String) at ./client.jl:439
   [14] top-level scope at none:6
   [15] eval(::Module, ::Any) at ./boot.jl:331
   [16] exec_options(::Base.JLOptions) at ./client.jl:264
   [17] _start() at ./client.jl:484
  
Uppercase Columns: Error During Test at /home/pkgeval/.julia/packages/LibPQ/PeS3p/test/runtests.jl:878
  Got exception outside of a @test
  could not connect to server: No such file or directory
  	Is the server running locally and accepting
  	connections on Unix domain socket "/tmp/.s.PGSQL.5432"?
  Stacktrace:
   [1] error(::Logger, ::LibPQ.Errors.PQConnectionError) at /home/pkgeval/.julia/packages/Memento/UgJr2/src/loggers.jl:439
   [2] handle_new_connection(::LibPQ.Connection; throw_error::Bool) at /home/pkgeval/.julia/packages/LibPQ/PeS3p/src/connections.jl:127
   [3] LibPQ.Connection(::String; throw_error::Bool, connect_timeout::Int64, options::Dict{String,String}, kwargs::Base.Iterators.Pairs{Union{},Union{},Tuple{},NamedTuple{(),Tuple{}}}) at /home/pkgeval/.julia/packages/LibPQ/PeS3p/src/connections.jl:320
   [4] top-level scope at /home/pkgeval/.julia/packages/LibPQ/PeS3p/test/runtests.jl:879
   [5] top-level scope at /buildworker/worker/package_linux64/build/usr/share/julia/stdlib/v1.5/Test/src/Test.jl:1113
   [6] top-level scope at /home/pkgeval/.julia/packages/LibPQ/PeS3p/test/runtests.jl:879
   [7] top-level scope at /buildworker/worker/package_linux64/build/usr/share/julia/stdlib/v1.5/Test/src/Test.jl:1113
   [8] top-level scope at /home/pkgeval/.julia/packages/LibPQ/PeS3p/test/runtests.jl:657
   [9] top-level scope at /buildworker/worker/package_linux64/build/usr/share/julia/stdlib/v1.5/Test/src/Test.jl:1113
   [10] top-level scope at /home/pkgeval/.julia/packages/LibPQ/PeS3p/test/runtests.jl:70
   [11] top-level scope at /buildworker/worker/package_linux64/build/usr/share/julia/stdlib/v1.5/Test/src/Test.jl:1113
   [12] top-level scope at /home/pkgeval/.julia/packages/LibPQ/PeS3p/test/runtests.jl:34
   [13] include(::String) at ./client.jl:439
   [14] top-level scope at none:6
   [15] eval(::Module, ::Any) at ./boot.jl:331
   [16] exec_options(::Base.JLOptions) at ./client.jl:264
   [17] _start() at ./client.jl:484
  
PQResultError: Error During Test at /home/pkgeval/.julia/packages/LibPQ/PeS3p/test/runtests.jl:896
  Got exception outside of a @test
  could not connect to server: No such file or directory
  	Is the server running locally and accepting
  	connections on Unix domain socket "/tmp/.s.PGSQL.5432"?
  Stacktrace:
   [1] error(::Logger, ::LibPQ.Errors.PQConnectionError) at /home/pkgeval/.julia/packages/Memento/UgJr2/src/loggers.jl:439
   [2] handle_new_connection(::LibPQ.Connection; throw_error::Bool) at /home/pkgeval/.julia/packages/LibPQ/PeS3p/src/connections.jl:127
   [3] LibPQ.Connection(::String; throw_error::Bool, connect_timeout::Int64, options::Dict{String,String}, kwargs::Base.Iterators.Pairs{Union{},Union{},Tuple{},NamedTuple{(),Tuple{}}}) at /home/pkgeval/.julia/packages/LibPQ/PeS3p/src/connections.jl:320
   [4] top-level scope at /home/pkgeval/.julia/packages/LibPQ/PeS3p/test/runtests.jl:897
   [5] top-level scope at /buildworker/worker/package_linux64/build/usr/share/julia/stdlib/v1.5/Test/src/Test.jl:1113
   [6] top-level scope at /home/pkgeval/.julia/packages/LibPQ/PeS3p/test/runtests.jl:897
   [7] top-level scope at /buildworker/worker/package_linux64/build/usr/share/julia/stdlib/v1.5/Test/src/Test.jl:1113
   [8] top-level scope at /home/pkgeval/.julia/packages/LibPQ/PeS3p/test/runtests.jl:657
   [9] top-level scope at /buildworker/worker/package_linux64/build/usr/share/julia/stdlib/v1.5/Test/src/Test.jl:1113
   [10] top-level scope at /home/pkgeval/.julia/packages/LibPQ/PeS3p/test/runtests.jl:70
   [11] top-level scope at /buildworker/worker/package_linux64/build/usr/share/julia/stdlib/v1.5/Test/src/Test.jl:1113
   [12] top-level scope at /home/pkgeval/.julia/packages/LibPQ/PeS3p/test/runtests.jl:34
   [13] include(::String) at ./client.jl:439
   [14] top-level scope at none:6
   [15] eval(::Module, ::Any) at ./boot.jl:331
   [16] exec_options(::Base.JLOptions) at ./client.jl:264
   [17] _start() at ./client.jl:484
  
Automatic: Error During Test at /home/pkgeval/.julia/packages/LibPQ/PeS3p/test/runtests.jl:943
  Got exception outside of a @test
  could not connect to server: No such file or directory
  	Is the server running locally and accepting
  	connections on Unix domain socket "/tmp/.s.PGSQL.5432"?
  Stacktrace:
   [1] error(::Logger, ::LibPQ.Errors.PQConnectionError) at /home/pkgeval/.julia/packages/Memento/UgJr2/src/loggers.jl:439
   [2] handle_new_connection(::LibPQ.Connection; throw_error::Bool) at /home/pkgeval/.julia/packages/LibPQ/PeS3p/src/connections.jl:127
   [3] LibPQ.Connection(::String; throw_error::Bool, connect_timeout::Int64, options::Dict{String,String}, kwargs::Base.Iterators.Pairs{Union{},Union{},Tuple{},NamedTuple{(),Tuple{}}}) at /home/pkgeval/.julia/packages/LibPQ/PeS3p/src/connections.jl:320
   [4] top-level scope at /home/pkgeval/.julia/packages/LibPQ/PeS3p/test/runtests.jl:944
   [5] top-level scope at /buildworker/worker/package_linux64/build/usr/share/julia/stdlib/v1.5/Test/src/Test.jl:1113
   [6] top-level scope at /home/pkgeval/.julia/packages/LibPQ/PeS3p/test/runtests.jl:944
   [7] top-level scope at /buildworker/worker/package_linux64/build/usr/share/julia/stdlib/v1.5/Test/src/Test.jl:1113
   [8] top-level scope at /home/pkgeval/.julia/packages/LibPQ/PeS3p/test/runtests.jl:943
   [9] top-level scope at /buildworker/worker/package_linux64/build/usr/share/julia/stdlib/v1.5/Test/src/Test.jl:1113
   [10] top-level scope at /home/pkgeval/.julia/packages/LibPQ/PeS3p/test/runtests.jl:657
   [11] top-level scope at /buildworker/worker/package_linux64/build/usr/share/julia/stdlib/v1.5/Test/src/Test.jl:1113
   [12] top-level scope at /home/pkgeval/.julia/packages/LibPQ/PeS3p/test/runtests.jl:70
   [13] top-level scope at /buildworker/worker/package_linux64/build/usr/share/julia/stdlib/v1.5/Test/src/Test.jl:1113
   [14] top-level scope at /home/pkgeval/.julia/packages/LibPQ/PeS3p/test/runtests.jl:34
   [15] include(::String) at ./client.jl:439
   [16] top-level scope at none:6
   [17] eval(::Module, ::Any) at ./boot.jl:331
   [18] exec_options(::Base.JLOptions) at ./client.jl:264
   [19] _start() at ./client.jl:484
  
Overrides: Error During Test at /home/pkgeval/.julia/packages/LibPQ/PeS3p/test/runtests.jl:974
  Got exception outside of a @test
  could not connect to server: No such file or directory
  	Is the server running locally and accepting
  	connections on Unix domain socket "/tmp/.s.PGSQL.5432"?
  Stacktrace:
   [1] error(::Logger, ::LibPQ.Errors.PQConnectionError) at /home/pkgeval/.julia/packages/Memento/UgJr2/src/loggers.jl:439
   [2] handle_new_connection(::LibPQ.Connection; throw_error::Bool) at /home/pkgeval/.julia/packages/LibPQ/PeS3p/src/connections.jl:127
   [3] LibPQ.Connection(::String; throw_error::Bool, connect_timeout::Int64, options::Dict{String,String}, kwargs::Base.Iterators.Pairs{Union{},Union{},Tuple{},NamedTuple{(),Tuple{}}}) at /home/pkgeval/.julia/packages/LibPQ/PeS3p/src/connections.jl:320
   [4] top-level scope at /home/pkgeval/.julia/packages/LibPQ/PeS3p/test/runtests.jl:975
   [5] top-level scope at /buildworker/worker/package_linux64/build/usr/share/julia/stdlib/v1.5/Test/src/Test.jl:1113
   [6] top-level scope at /home/pkgeval/.julia/packages/LibPQ/PeS3p/test/runtests.jl:975
   [7] top-level scope at /buildworker/worker/package_linux64/build/usr/share/julia/stdlib/v1.5/Test/src/Test.jl:1113
   [8] top-level scope at /home/pkgeval/.julia/packages/LibPQ/PeS3p/test/runtests.jl:943
   [9] top-level scope at /buildworker/worker/package_linux64/build/usr/share/julia/stdlib/v1.5/Test/src/Test.jl:1113
   [10] top-level scope at /home/pkgeval/.julia/packages/LibPQ/PeS3p/test/runtests.jl:657
   [11] top-level scope at /buildworker/worker/package_linux64/build/usr/share/julia/stdlib/v1.5/Test/src/Test.jl:1113
   [12] top-level scope at /home/pkgeval/.julia/packages/LibPQ/PeS3p/test/runtests.jl:70
   [13] top-level scope at /buildworker/worker/package_linux64/build/usr/share/julia/stdlib/v1.5/Test/src/Test.jl:1113
   [14] top-level scope at /home/pkgeval/.julia/packages/LibPQ/PeS3p/test/runtests.jl:34
   [15] include(::String) at ./client.jl:439
   [16] top-level scope at none:6
   [17] eval(::Module, ::Any) at ./boot.jl:331
   [18] exec_options(::Base.JLOptions) at ./client.jl:264
   [19] _start() at ./client.jl:484
  
Default Types: Error During Test at /home/pkgeval/.julia/packages/LibPQ/PeS3p/test/runtests.jl:1013
  Got exception outside of a @test
  could not connect to server: No such file or directory
  	Is the server running locally and accepting
  	connections on Unix domain socket "/tmp/.s.PGSQL.5432"?
  Stacktrace:
   [1] error(::Logger, ::LibPQ.Errors.PQConnectionError) at /home/pkgeval/.julia/packages/Memento/UgJr2/src/loggers.jl:439
   [2] handle_new_connection(::LibPQ.Connection; throw_error::Bool) at /home/pkgeval/.julia/packages/LibPQ/PeS3p/src/connections.jl:127
   [3] LibPQ.Connection(::String; throw_error::Bool, connect_timeout::Int64, options::Dict{String,String}, kwargs::Base.Iterators.Pairs{Union{},Union{},Tuple{},NamedTuple{(),Tuple{}}}) at /home/pkgeval/.julia/packages/LibPQ/PeS3p/src/connections.jl:320
   [4] top-level scope at /home/pkgeval/.julia/packages/LibPQ/PeS3p/test/runtests.jl:1014
   [5] top-level scope at /buildworker/worker/package_linux64/build/usr/share/julia/stdlib/v1.5/Test/src/Test.jl:1113
   [6] top-level scope at /home/pkgeval/.julia/packages/LibPQ/PeS3p/test/runtests.jl:1014
   [7] top-level scope at /buildworker/worker/package_linux64/build/usr/share/julia/stdlib/v1.5/Test/src/Test.jl:1113
   [8] top-level scope at /home/pkgeval/.julia/packages/LibPQ/PeS3p/test/runtests.jl:1013
   [9] top-level scope at /buildworker/worker/package_linux64/build/usr/share/julia/stdlib/v1.5/Test/src/Test.jl:1113
   [10] top-level scope at /home/pkgeval/.julia/packages/LibPQ/PeS3p/test/runtests.jl:943
   [11] top-level scope at /buildworker/worker/package_linux64/build/usr/share/julia/stdlib/v1.5/Test/src/Test.jl:1113
   [12] top-level scope at /home/pkgeval/.julia/packages/LibPQ/PeS3p/test/runtests.jl:657
   [13] top-level scope at /buildworker/worker/package_linux64/build/usr/share/julia/stdlib/v1.5/Test/src/Test.jl:1113
   [14] top-level scope at /home/pkgeval/.julia/packages/LibPQ/PeS3p/test/runtests.jl:70
   [15] top-level scope at /buildworker/worker/package_linux64/build/usr/share/julia/stdlib/v1.5/Test/src/Test.jl:1113
   [16] top-level scope at /home/pkgeval/.julia/packages/LibPQ/PeS3p/test/runtests.jl:34
   [17] include(::String) at ./client.jl:439
   [18] top-level scope at none:6
   [19] eval(::Module, ::Any) at ./boot.jl:331
   [20] exec_options(::Base.JLOptions) at ./client.jl:264
   [21] _start() at ./client.jl:484
  
Specified Types: Error During Test at /home/pkgeval/.julia/packages/LibPQ/PeS3p/test/runtests.jl:1125
  Got exception outside of a @test
  could not connect to server: No such file or directory
  	Is the server running locally and accepting
  	connections on Unix domain socket "/tmp/.s.PGSQL.5432"?
  Stacktrace:
   [1] error(::Logger, ::LibPQ.Errors.PQConnectionError) at /home/pkgeval/.julia/packages/Memento/UgJr2/src/loggers.jl:439
   [2] handle_new_connection(::LibPQ.Connection; throw_error::Bool) at /home/pkgeval/.julia/packages/LibPQ/PeS3p/src/connections.jl:127
   [3] LibPQ.Connection(::String; throw_error::Bool, connect_timeout::Int64, options::Dict{String,String}, kwargs::Base.Iterators.Pairs{Union{},Union{},Tuple{},NamedTuple{(),Tuple{}}}) at /home/pkgeval/.julia/packages/LibPQ/PeS3p/src/connections.jl:320
   [4] top-level scope at /home/pkgeval/.julia/packages/LibPQ/PeS3p/test/runtests.jl:1126
   [5] top-level scope at /buildworker/worker/package_linux64/build/usr/share/julia/stdlib/v1.5/Test/src/Test.jl:1113
   [6] top-level scope at /home/pkgeval/.julia/packages/LibPQ/PeS3p/test/runtests.jl:1126
   [7] top-level scope at /buildworker/worker/package_linux64/build/usr/share/julia/stdlib/v1.5/Test/src/Test.jl:1113
   [8] top-level scope at /home/pkgeval/.julia/packages/LibPQ/PeS3p/test/runtests.jl:1013
   [9] top-level scope at /buildworker/worker/package_linux64/build/usr/share/julia/stdlib/v1.5/Test/src/Test.jl:1113
   [10] top-level scope at /home/pkgeval/.julia/packages/LibPQ/PeS3p/test/runtests.jl:943
   [11] top-level scope at /buildworker/worker/package_linux64/build/usr/share/julia/stdlib/v1.5/Test/src/Test.jl:1113
   [12] top-level scope at /home/pkgeval/.julia/packages/LibPQ/PeS3p/test/runtests.jl:657
   [13] top-level scope at /buildworker/worker/package_linux64/build/usr/share/julia/stdlib/v1.5/Test/src/Test.jl:1113
   [14] top-level scope at /home/pkgeval/.julia/packages/LibPQ/PeS3p/test/runtests.jl:70
   [15] top-level scope at /buildworker/worker/package_linux64/build/usr/share/julia/stdlib/v1.5/Test/src/Test.jl:1113
   [16] top-level scope at /home/pkgeval/.julia/packages/LibPQ/PeS3p/test/runtests.jl:34
   [17] include(::String) at ./client.jl:439
   [18] top-level scope at none:6
   [19] eval(::Module, ::Any) at ./boot.jl:331
   [20] exec_options(::Base.JLOptions) at ./client.jl:264
   [21] _start() at ./client.jl:484
  
Parameters: Error During Test at /home/pkgeval/.julia/packages/LibPQ/PeS3p/test/runtests.jl:1187
  Got exception outside of a @test
  could not connect to server: No such file or directory
  	Is the server running locally and accepting
  	connections on Unix domain socket "/tmp/.s.PGSQL.5432"?
  Stacktrace:
   [1] error(::Logger, ::LibPQ.Errors.PQConnectionError) at /home/pkgeval/.julia/packages/Memento/UgJr2/src/loggers.jl:439
   [2] handle_new_connection(::LibPQ.Connection; throw_error::Bool) at /home/pkgeval/.julia/packages/LibPQ/PeS3p/src/connections.jl:127
   [3] LibPQ.Connection(::String; throw_error::Bool, connect_timeout::Int64, options::Dict{String,String}, kwargs::Base.Iterators.Pairs{Union{},Union{},Tuple{},NamedTuple{(),Tuple{}}}) at /home/pkgeval/.julia/packages/LibPQ/PeS3p/src/connections.jl:320
   [4] top-level scope at /home/pkgeval/.julia/packages/LibPQ/PeS3p/test/runtests.jl:1188
   [5] top-level scope at /buildworker/worker/package_linux64/build/usr/share/julia/stdlib/v1.5/Test/src/Test.jl:1113
   [6] top-level scope at /home/pkgeval/.julia/packages/LibPQ/PeS3p/test/runtests.jl:1188
   [7] top-level scope at /buildworker/worker/package_linux64/build/usr/share/julia/stdlib/v1.5/Test/src/Test.jl:1113
   [8] top-level scope at /home/pkgeval/.julia/packages/LibPQ/PeS3p/test/runtests.jl:657
   [9] top-level scope at /buildworker/worker/package_linux64/build/usr/share/julia/stdlib/v1.5/Test/src/Test.jl:1113
   [10] top-level scope at /home/pkgeval/.julia/packages/LibPQ/PeS3p/test/runtests.jl:70
   [11] top-level scope at /buildworker/worker/package_linux64/build/usr/share/julia/stdlib/v1.5/Test/src/Test.jl:1113
   [12] top-level scope at /home/pkgeval/.julia/packages/LibPQ/PeS3p/test/runtests.jl:34
   [13] include(::String) at ./client.jl:439
   [14] top-level scope at none:6
   [15] eval(::Module, ::Any) at ./boot.jl:331
   [16] exec_options(::Base.JLOptions) at ./client.jl:264
   [17] _start() at ./client.jl:484
  
Syntax Errors: Error During Test at /home/pkgeval/.julia/packages/LibPQ/PeS3p/test/runtests.jl:1222
  Got exception outside of a @test
  could not connect to server: No such file or directory
  	Is the server running locally and accepting
  	connections on Unix domain socket "/tmp/.s.PGSQL.5432"?
  Stacktrace:
   [1] error(::Logger, ::LibPQ.Errors.PQConnectionError) at /home/pkgeval/.julia/packages/Memento/UgJr2/src/loggers.jl:439
   [2] handle_new_connection(::LibPQ.Connection; throw_error::Bool) at /home/pkgeval/.julia/packages/LibPQ/PeS3p/src/connections.jl:127
   [3] LibPQ.Connection(::String; throw_error::Bool, connect_timeout::Int64, options::Dict{String,String}, kwargs::Base.Iterators.Pairs{Union{},Union{},Tuple{},NamedTuple{(),Tuple{}}}) at /home/pkgeval/.julia/packages/LibPQ/PeS3p/src/connections.jl:320
   [4] top-level scope at /home/pkgeval/.julia/packages/LibPQ/PeS3p/test/runtests.jl:1223
   [5] top-level scope at /buildworker/worker/package_linux64/build/usr/share/julia/stdlib/v1.5/Test/src/Test.jl:1113
   [6] top-level scope at /home/pkgeval/.julia/packages/LibPQ/PeS3p/test/runtests.jl:1223
   [7] top-level scope at /buildworker/worker/package_linux64/build/usr/share/julia/stdlib/v1.5/Test/src/Test.jl:1113
   [8] top-level scope at /home/pkgeval/.julia/packages/LibPQ/PeS3p/test/runtests.jl:1222
   [9] top-level scope at /buildworker/worker/package_linux64/build/usr/share/julia/stdlib/v1.5/Test/src/Test.jl:1113
   [10] top-level scope at /home/pkgeval/.julia/packages/LibPQ/PeS3p/test/runtests.jl:657
   [11] top-level scope at /buildworker/worker/package_linux64/build/usr/share/julia/stdlib/v1.5/Test/src/Test.jl:1113
   [12] top-level scope at /home/pkgeval/.julia/packages/LibPQ/PeS3p/test/runtests.jl:70
   [13] top-level scope at /buildworker/worker/package_linux64/build/usr/share/julia/stdlib/v1.5/Test/src/Test.jl:1113
   [14] top-level scope at /home/pkgeval/.julia/packages/LibPQ/PeS3p/test/runtests.jl:34
   [15] include(::String) at ./client.jl:439
   [16] top-level scope at none:6
   [17] eval(::Module, ::Any) at ./boot.jl:331
   [18] exec_options(::Base.JLOptions) at ./client.jl:264
   [19] _start() at ./client.jl:484
  
Wrong No. Parameters: Error During Test at /home/pkgeval/.julia/packages/LibPQ/PeS3p/test/runtests.jl:1240
  Got exception outside of a @test
  could not connect to server: No such file or directory
  	Is the server running locally and accepting
  	connections on Unix domain socket "/tmp/.s.PGSQL.5432"?
  Stacktrace:
   [1] error(::Logger, ::LibPQ.Errors.PQConnectionError) at /home/pkgeval/.julia/packages/Memento/UgJr2/src/loggers.jl:439
   [2] handle_new_connection(::LibPQ.Connection; throw_error::Bool) at /home/pkgeval/.julia/packages/LibPQ/PeS3p/src/connections.jl:127
   [3] LibPQ.Connection(::String; throw_error::Bool, connect_timeout::Int64, options::Dict{String,String}, kwargs::Base.Iterators.Pairs{Union{},Union{},Tuple{},NamedTuple{(),Tuple{}}}) at /home/pkgeval/.julia/packages/LibPQ/PeS3p/src/connections.jl:320
   [4] top-level scope at /home/pkgeval/.julia/packages/LibPQ/PeS3p/test/runtests.jl:1241
   [5] top-level scope at /buildworker/worker/package_linux64/build/usr/share/julia/stdlib/v1.5/Test/src/Test.jl:1113
   [6] top-level scope at /home/pkgeval/.julia/packages/LibPQ/PeS3p/test/runtests.jl:1241
   [7] top-level scope at /buildworker/worker/package_linux64/build/usr/share/julia/stdlib/v1.5/Test/src/Test.jl:1113
   [8] top-level scope at /home/pkgeval/.julia/packages/LibPQ/PeS3p/test/runtests.jl:1222
   [9] top-level scope at /buildworker/worker/package_linux64/build/usr/share/julia/stdlib/v1.5/Test/src/Test.jl:1113
   [10] top-level scope at /home/pkgeval/.julia/packages/LibPQ/PeS3p/test/runtests.jl:657
   [11] top-level scope at /buildworker/worker/package_linux64/build/usr/share/julia/stdlib/v1.5/Test/src/Test.jl:1113
   [12] top-level scope at /home/pkgeval/.julia/packages/LibPQ/PeS3p/test/runtests.jl:70
   [13] top-level scope at /buildworker/worker/package_linux64/build/usr/share/julia/stdlib/v1.5/Test/src/Test.jl:1113
   [14] top-level scope at /home/pkgeval/.julia/packages/LibPQ/PeS3p/test/runtests.jl:34
   [15] include(::String) at ./client.jl:439
   [16] top-level scope at none:6
   [17] eval(::Module, ::Any) at ./boot.jl:331
   [18] exec_options(::Base.JLOptions) at ./client.jl:264
   [19] _start() at ./client.jl:484
  
Interface Errors: Error During Test at /home/pkgeval/.julia/packages/LibPQ/PeS3p/test/runtests.jl:1264
  Got exception outside of a @test
  could not connect to server: No such file or directory
  	Is the server running locally and accepting
  	connections on Unix domain socket "/tmp/.s.PGSQL.5432"?
  Stacktrace:
   [1] error(::Logger, ::LibPQ.Errors.PQConnectionError) at /home/pkgeval/.julia/packages/Memento/UgJr2/src/loggers.jl:439
   [2] handle_new_connection(::LibPQ.Connection; throw_error::Bool) at /home/pkgeval/.julia/packages/LibPQ/PeS3p/src/connections.jl:127
   [3] LibPQ.Connection(::String; throw_error::Bool, connect_timeout::Int64, options::Dict{String,String}, kwargs::Base.Iterators.Pairs{Union{},Union{},Tuple{},NamedTuple{(),Tuple{}}}) at /home/pkgeval/.julia/packages/LibPQ/PeS3p/src/connections.jl:320
   [4] top-level scope at /home/pkgeval/.julia/packages/LibPQ/PeS3p/test/runtests.jl:1265
   [5] top-level scope at /buildworker/worker/package_linux64/build/usr/share/julia/stdlib/v1.5/Test/src/Test.jl:1113
   [6] top-level scope at /home/pkgeval/.julia/packages/LibPQ/PeS3p/test/runtests.jl:1265
   [7] top-level scope at /buildworker/worker/package_linux64/build/usr/share/julia/stdlib/v1.5/Test/src/Test.jl:1113
   [8] top-level scope at /home/pkgeval/.julia/packages/LibPQ/PeS3p/test/runtests.jl:657
   [9] top-level scope at /buildworker/worker/package_linux64/build/usr/share/julia/stdlib/v1.5/Test/src/Test.jl:1113
   [10] top-level scope at /home/pkgeval/.julia/packages/LibPQ/PeS3p/test/runtests.jl:70
   [11] top-level scope at /buildworker/worker/package_linux64/build/usr/share/julia/stdlib/v1.5/Test/src/Test.jl:1113
   [12] top-level scope at /home/pkgeval/.julia/packages/LibPQ/PeS3p/test/runtests.jl:34
   [13] include(::String) at ./client.jl:439
   [14] top-level scope at none:6
   [15] eval(::Module, ::Any) at ./boot.jl:331
   [16] exec_options(::Base.JLOptions) at ./client.jl:264
   [17] _start() at ./client.jl:484
  
No Params, Output: Error During Test at /home/pkgeval/.julia/packages/LibPQ/PeS3p/test/runtests.jl:1281
  Got exception outside of a @test
  could not connect to server: No such file or directory
  	Is the server running locally and accepting
  	connections on Unix domain socket "/tmp/.s.PGSQL.5432"?
  Stacktrace:
   [1] error(::Logger, ::LibPQ.Errors.PQConnectionError) at /home/pkgeval/.julia/packages/Memento/UgJr2/src/loggers.jl:439
   [2] handle_new_connection(::LibPQ.Connection; throw_error::Bool) at /home/pkgeval/.julia/packages/LibPQ/PeS3p/src/connections.jl:127
   [3] LibPQ.Connection(::String; throw_error::Bool, connect_timeout::Int64, options::Dict{String,String}, kwargs::Base.Iterators.Pairs{Union{},Union{},Tuple{},NamedTuple{(),Tuple{}}}) at /home/pkgeval/.julia/packages/LibPQ/PeS3p/src/connections.jl:320
   [4] top-level scope at /home/pkgeval/.julia/packages/LibPQ/PeS3p/test/runtests.jl:1282
   [5] top-level scope at /buildworker/worker/package_linux64/build/usr/share/julia/stdlib/v1.5/Test/src/Test.jl:1113
   [6] top-level scope at /home/pkgeval/.julia/packages/LibPQ/PeS3p/test/runtests.jl:1282
   [7] top-level scope at /buildworker/worker/package_linux64/build/usr/share/julia/stdlib/v1.5/Test/src/Test.jl:1113
   [8] top-level scope at /home/pkgeval/.julia/packages/LibPQ/PeS3p/test/runtests.jl:1281
   [9] top-level scope at /buildworker/worker/package_linux64/build/usr/share/julia/stdlib/v1.5/Test/src/Test.jl:1113
   [10] top-level scope at /home/pkgeval/.julia/packages/LibPQ/PeS3p/test/runtests.jl:70
   [11] top-level scope at /buildworker/worker/package_linux64/build/usr/share/julia/stdlib/v1.5/Test/src/Test.jl:1113
   [12] top-level scope at /home/pkgeval/.julia/packages/LibPQ/PeS3p/test/runtests.jl:34
   [13] include(::String) at ./client.jl:439
   [14] top-level scope at none:6
   [15] eval(::Module, ::Any) at ./boot.jl:331
   [16] exec_options(::Base.JLOptions) at ./client.jl:264
   [17] _start() at ./client.jl:484
  
Params, Output: Error During Test at /home/pkgeval/.julia/packages/LibPQ/PeS3p/test/runtests.jl:1302
  Got exception outside of a @test
  could not connect to server: No such file or directory
  	Is the server running locally and accepting
  	connections on Unix domain socket "/tmp/.s.PGSQL.5432"?
  Stacktrace:
   [1] error(::Logger, ::LibPQ.Errors.PQConnectionError) at /home/pkgeval/.julia/packages/Memento/UgJr2/src/loggers.jl:439
   [2] handle_new_connection(::LibPQ.Connection; throw_error::Bool) at /home/pkgeval/.julia/packages/LibPQ/PeS3p/src/connections.jl:127
   [3] LibPQ.Connection(::String; throw_error::Bool, connect_timeout::Int64, options::Dict{String,String}, kwargs::Base.Iterators.Pairs{Union{},Union{},Tuple{},NamedTuple{(),Tuple{}}}) at /home/pkgeval/.julia/packages/LibPQ/PeS3p/src/connections.jl:320
   [4] top-level scope at /home/pkgeval/.julia/packages/LibPQ/PeS3p/test/runtests.jl:1303
   [5] top-level scope at /buildworker/worker/package_linux64/build/usr/share/julia/stdlib/v1.5/Test/src/Test.jl:1113
   [6] top-level scope at /home/pkgeval/.julia/packages/LibPQ/PeS3p/test/runtests.jl:1303
   [7] top-level scope at /buildworker/worker/package_linux64/build/usr/share/julia/stdlib/v1.5/Test/src/Test.jl:1113
   [8] top-level scope at /home/pkgeval/.julia/packages/LibPQ/PeS3p/test/runtests.jl:1281
   [9] top-level scope at /buildworker/worker/package_linux64/build/usr/share/julia/stdlib/v1.5/Test/src/Test.jl:1113
   [10] top-level scope at /home/pkgeval/.julia/packages/LibPQ/PeS3p/test/runtests.jl:70
   [11] top-level scope at /buildworker/worker/package_linux64/build/usr/share/julia/stdlib/v1.5/Test/src/Test.jl:1113
   [12] top-level scope at /home/pkgeval/.julia/packages/LibPQ/PeS3p/test/runtests.jl:34
   [13] include(::String) at ./client.jl:439
   [14] top-level scope at none:6
   [15] eval(::Module, ::Any) at ./boot.jl:331
   [16] exec_options(::Base.JLOptions) at ./client.jl:264
   [17] _start() at ./client.jl:484
  
Basic: Error During Test at /home/pkgeval/.julia/packages/LibPQ/PeS3p/test/runtests.jl:1331
  Got exception outside of a @test
  could not connect to server: No such file or directory
  	Is the server running locally and accepting
  	connections on Unix domain socket "/tmp/.s.PGSQL.5432"?
  Stacktrace:
   [1] error(::Logger, ::LibPQ.Errors.PQConnectionError) at /home/pkgeval/.julia/packages/Memento/UgJr2/src/loggers.jl:439
   [2] handle_new_connection(::LibPQ.Connection; throw_error::Bool) at /home/pkgeval/.julia/packages/LibPQ/PeS3p/src/connections.jl:127
   [3] LibPQ.Connection(::String; throw_error::Bool, connect_timeout::Int64, options::Dict{String,String}, kwargs::Base.Iterators.Pairs{Union{},Union{},Tuple{},NamedTuple{(),Tuple{}}}) at /home/pkgeval/.julia/packages/LibPQ/PeS3p/src/connections.jl:320
   [4] top-level scope at /home/pkgeval/.julia/packages/LibPQ/PeS3p/test/runtests.jl:1332
   [5] top-level scope at /buildworker/worker/package_linux64/build/usr/share/julia/stdlib/v1.5/Test/src/Test.jl:1113
   [6] top-level scope at /home/pkgeval/.julia/packages/LibPQ/PeS3p/test/runtests.jl:1332
   [7] top-level scope at /buildworker/worker/package_linux64/build/usr/share/julia/stdlib/v1.5/Test/src/Test.jl:1113
   [8] top-level scope at /home/pkgeval/.julia/packages/LibPQ/PeS3p/test/runtests.jl:1329
   [9] top-level scope at /buildworker/worker/package_linux64/build/usr/share/julia/stdlib/v1.5/Test/src/Test.jl:1113
   [10] top-level scope at /home/pkgeval/.julia/packages/LibPQ/PeS3p/test/runtests.jl:70
   [11] top-level scope at /buildworker/worker/package_linux64/build/usr/share/julia/stdlib/v1.5/Test/src/Test.jl:1113
   [12] top-level scope at /home/pkgeval/.julia/packages/LibPQ/PeS3p/test/runtests.jl:34
   [13] include(::String) at ./client.jl:439
   [14] top-level scope at none:6
   [15] eval(::Module, ::Any) at ./boot.jl:331
   [16] exec_options(::Base.JLOptions) at ./client.jl:264
   [17] _start() at ./client.jl:484
  
Parameters: Error During Test at /home/pkgeval/.julia/packages/LibPQ/PeS3p/test/runtests.jl:1353
  Got exception outside of a @test
  could not connect to server: No such file or directory
  	Is the server running locally and accepting
  	connections on Unix domain socket "/tmp/.s.PGSQL.5432"?
  Stacktrace:
   [1] error(::Logger, ::LibPQ.Errors.PQConnectionError) at /home/pkgeval/.julia/packages/Memento/UgJr2/src/loggers.jl:439
   [2] handle_new_connection(::LibPQ.Connection; throw_error::Bool) at /home/pkgeval/.julia/packages/LibPQ/PeS3p/src/connections.jl:127
   [3] LibPQ.Connection(::String; throw_error::Bool, connect_timeout::Int64, options::Dict{String,String}, kwargs::Base.Iterators.Pairs{Union{},Union{},Tuple{},NamedTuple{(),Tuple{}}}) at /home/pkgeval/.julia/packages/LibPQ/PeS3p/src/connections.jl:320
   [4] top-level scope at /home/pkgeval/.julia/packages/LibPQ/PeS3p/test/runtests.jl:1354
   [5] top-level scope at /buildworker/worker/package_linux64/build/usr/share/julia/stdlib/v1.5/Test/src/Test.jl:1113
   [6] top-level scope at /home/pkgeval/.julia/packages/LibPQ/PeS3p/test/runtests.jl:1354
   [7] top-level scope at /buildworker/worker/package_linux64/build/usr/share/julia/stdlib/v1.5/Test/src/Test.jl:1113
   [8] top-level scope at /home/pkgeval/.julia/packages/LibPQ/PeS3p/test/runtests.jl:1329
   [9] top-level scope at /buildworker/worker/package_linux64/build/usr/share/julia/stdlib/v1.5/Test/src/Test.jl:1113
   [10] top-level scope at /home/pkgeval/.julia/packages/LibPQ/PeS3p/test/runtests.jl:70
   [11] top-level scope at /buildworker/worker/package_linux64/build/usr/share/julia/stdlib/v1.5/Test/src/Test.jl:1113
   [12] top-level scope at /home/pkgeval/.julia/packages/LibPQ/PeS3p/test/runtests.jl:34
   [13] include(::String) at ./client.jl:439
   [14] top-level scope at none:6
   [15] eval(::Module, ::Any) at ./boot.jl:331
   [16] exec_options(::Base.JLOptions) at ./client.jl:264
   [17] _start() at ./client.jl:484
  
Wait in line to complete: Error During Test at /home/pkgeval/.julia/packages/LibPQ/PeS3p/test/runtests.jl:1385
  Got exception outside of a @test
  could not connect to server: No such file or directory
  	Is the server running locally and accepting
  	connections on Unix domain socket "/tmp/.s.PGSQL.5432"?
  Stacktrace:
   [1] error(::Logger, ::LibPQ.Errors.PQConnectionError) at /home/pkgeval/.julia/packages/Memento/UgJr2/src/loggers.jl:439
   [2] handle_new_connection(::LibPQ.Connection; throw_error::Bool) at /home/pkgeval/.julia/packages/LibPQ/PeS3p/src/connections.jl:127
   [3] LibPQ.Connection(::String; throw_error::Bool, connect_timeout::Int64, options::Dict{String,String}, kwargs::Base.Iterators.Pairs{Union{},Union{},Tuple{},NamedTuple{(),Tuple{}}}) at /home/pkgeval/.julia/packages/LibPQ/PeS3p/src/connections.jl:320
   [4] top-level scope at /home/pkgeval/.julia/packages/LibPQ/PeS3p/test/runtests.jl:1386
   [5] top-level scope at /buildworker/worker/package_linux64/build/usr/share/julia/stdlib/v1.5/Test/src/Test.jl:1113
   [6] top-level scope at /home/pkgeval/.julia/packages/LibPQ/PeS3p/test/runtests.jl:1386
   [7] top-level scope at /buildworker/worker/package_linux64/build/usr/share/julia/stdlib/v1.5/Test/src/Test.jl:1113
   [8] top-level scope at /home/pkgeval/.julia/packages/LibPQ/PeS3p/test/runtests.jl:1329
   [9] top-level scope at /buildworker/worker/package_linux64/build/usr/share/julia/stdlib/v1.5/Test/src/Test.jl:1113
   [10] top-level scope at /home/pkgeval/.julia/packages/LibPQ/PeS3p/test/runtests.jl:70
   [11] top-level scope at /buildworker/worker/package_linux64/build/usr/share/julia/stdlib/v1.5/Test/src/Test.jl:1113
   [12] top-level scope at /home/pkgeval/.julia/packages/LibPQ/PeS3p/test/runtests.jl:34
   [13] include(::String) at ./client.jl:439
   [14] top-level scope at none:6
   [15] eval(::Module, ::Any) at ./boot.jl:331
   [16] exec_options(::Base.JLOptions) at ./client.jl:264
   [17] _start() at ./client.jl:484
  
Cancel: Error During Test at /home/pkgeval/.julia/packages/LibPQ/PeS3p/test/runtests.jl:1414
  Got exception outside of a @test
  could not connect to server: No such file or directory
  	Is the server running locally and accepting
  	connections on Unix domain socket "/tmp/.s.PGSQL.5432"?
  Stacktrace:
   [1] error(::Logger, ::LibPQ.Errors.PQConnectionError) at /home/pkgeval/.julia/packages/Memento/UgJr2/src/loggers.jl:439
   [2] handle_new_connection(::LibPQ.Connection; throw_error::Bool) at /home/pkgeval/.julia/packages/LibPQ/PeS3p/src/connections.jl:127
   [3] LibPQ.Connection(::String; throw_error::Bool, connect_timeout::Int64, options::Dict{String,String}, kwargs::Base.Iterators.Pairs{Union{},Union{},Tuple{},NamedTuple{(),Tuple{}}}) at /home/pkgeval/.julia/packages/LibPQ/PeS3p/src/connections.jl:320
   [4] top-level scope at /home/pkgeval/.julia/packages/LibPQ/PeS3p/test/runtests.jl:1415
   [5] top-level scope at /buildworker/worker/package_linux64/build/usr/share/julia/stdlib/v1.5/Test/src/Test.jl:1113
   [6] top-level scope at /home/pkgeval/.julia/packages/LibPQ/PeS3p/test/runtests.jl:1415
   [7] top-level scope at /buildworker/worker/package_linux64/build/usr/share/julia/stdlib/v1.5/Test/src/Test.jl:1113
   [8] top-level scope at /home/pkgeval/.julia/packages/LibPQ/PeS3p/test/runtests.jl:1329
   [9] top-level scope at /buildworker/worker/package_linux64/build/usr/share/julia/stdlib/v1.5/Test/src/Test.jl:1113
   [10] top-level scope at /home/pkgeval/.julia/packages/LibPQ/PeS3p/test/runtests.jl:70
   [11] top-level scope at /buildworker/worker/package_linux64/build/usr/share/julia/stdlib/v1.5/Test/src/Test.jl:1113
   [12] top-level scope at /home/pkgeval/.julia/packages/LibPQ/PeS3p/test/runtests.jl:34
   [13] include(::String) at ./client.jl:439
   [14] top-level scope at none:6
   [15] eval(::Module, ::Any) at ./boot.jl:331
   [16] exec_options(::Base.JLOptions) at ./client.jl:264
   [17] _start() at ./client.jl:484
  
Canceled by closing connection: Error During Test at /home/pkgeval/.julia/packages/LibPQ/PeS3p/test/runtests.jl:1446
  Got exception outside of a @test
  could not connect to server: No such file or directory
  	Is the server running locally and accepting
  	connections on Unix domain socket "/tmp/.s.PGSQL.5432"?
  Stacktrace:
   [1] error(::Logger, ::LibPQ.Errors.PQConnectionError) at /home/pkgeval/.julia/packages/Memento/UgJr2/src/loggers.jl:439
   [2] handle_new_connection(::LibPQ.Connection; throw_error::Bool) at /home/pkgeval/.julia/packages/LibPQ/PeS3p/src/connections.jl:127
   [3] LibPQ.Connection(::String; throw_error::Bool, connect_timeout::Int64, options::Dict{String,String}, kwargs::Base.Iterators.Pairs{Union{},Union{},Tuple{},NamedTuple{(),Tuple{}}}) at /home/pkgeval/.julia/packages/LibPQ/PeS3p/src/connections.jl:320
   [4] top-level scope at /home/pkgeval/.julia/packages/LibPQ/PeS3p/test/runtests.jl:1447
   [5] top-level scope at /buildworker/worker/package_linux64/build/usr/share/julia/stdlib/v1.5/Test/src/Test.jl:1113
   [6] top-level scope at /home/pkgeval/.julia/packages/LibPQ/PeS3p/test/runtests.jl:1447
   [7] top-level scope at /buildworker/worker/package_linux64/build/usr/share/julia/stdlib/v1.5/Test/src/Test.jl:1113
   [8] top-level scope at /home/pkgeval/.julia/packages/LibPQ/PeS3p/test/runtests.jl:1329
   [9] top-level scope at /buildworker/worker/package_linux64/build/usr/share/julia/stdlib/v1.5/Test/src/Test.jl:1113
   [10] top-level scope at /home/pkgeval/.julia/packages/LibPQ/PeS3p/test/runtests.jl:70
   [11] top-level scope at /buildworker/worker/package_linux64/build/usr/share/julia/stdlib/v1.5/Test/src/Test.jl:1113
   [12] top-level scope at /home/pkgeval/.julia/packages/LibPQ/PeS3p/test/runtests.jl:34
   [13] include(::String) at ./client.jl:439
   [14] top-level scope at none:6
   [15] eval(::Module, ::Any) at ./boot.jl:331
   [16] exec_options(::Base.JLOptions) at ./client.jl:264
   [17] _start() at ./client.jl:484
  
FDWatcher: bad file descriptor (EBADF): Error During Test at /home/pkgeval/.julia/packages/LibPQ/PeS3p/test/runtests.jl:1478
  Got exception outside of a @test
  could not connect to server: No such file or directory
  	Is the server running locally and accepting
  	connections on Unix domain socket "/tmp/.s.PGSQL.5432"?
  Stacktrace:
   [1] error(::Logger, ::LibPQ.Errors.PQConnectionError) at /home/pkgeval/.julia/packages/Memento/UgJr2/src/loggers.jl:439
   [2] handle_new_connection(::LibPQ.Connection; throw_error::Bool) at /home/pkgeval/.julia/packages/LibPQ/PeS3p/src/connections.jl:127
   [3] LibPQ.Connection(::String; throw_error::Bool, connect_timeout::Int64, options::Dict{String,String}, kwargs::Base.Iterators.Pairs{Union{},Union{},Tuple{},NamedTuple{(),Tuple{}}}) at /home/pkgeval/.julia/packages/LibPQ/PeS3p/src/connections.jl:320
   [4] top-level scope at /home/pkgeval/.julia/packages/LibPQ/PeS3p/test/runtests.jl:1479
   [5] top-level scope at /buildworker/worker/package_linux64/build/usr/share/julia/stdlib/v1.5/Test/src/Test.jl:1113
   [6] top-level scope at /home/pkgeval/.julia/packages/LibPQ/PeS3p/test/runtests.jl:1479
   [7] top-level scope at /buildworker/worker/package_linux64/build/usr/share/julia/stdlib/v1.5/Test/src/Test.jl:1113
   [8] top-level scope at /home/pkgeval/.julia/packages/LibPQ/PeS3p/test/runtests.jl:1329
   [9] top-level scope at /buildworker/worker/package_linux64/build/usr/share/julia/stdlib/v1.5/Test/src/Test.jl:1113
   [10] top-level scope at /home/pkgeval/.julia/packages/LibPQ/PeS3p/test/runtests.jl:70
   [11] top-level scope at /buildworker/worker/package_linux64/build/usr/share/julia/stdlib/v1.5/Test/src/Test.jl:1113
   [12] top-level scope at /home/pkgeval/.julia/packages/LibPQ/PeS3p/test/runtests.jl:34
   [13] include(::String) at ./client.jl:439
   [14] top-level scope at none:6
   [15] eval(::Module, ::Any) at ./boot.jl:331
   [16] exec_options(::Base.JLOptions) at ./client.jl:264
   [17] _start() at ./client.jl:484
  
Test Summary:                                | Pass  Fail  Error  Total
LibPQ                                        |   51    10     37     98
  ConninfoDisplay                            |    4                   4
  Version Numbers                            |   11                  11
  Online                                     |   36    10     37     83
    Example SELECT                           |    9    10      5     24
    Example INSERT and DELETE                |                 1      1
    COPY FROM                                |                 2      2
      Example COPY FROM                      |                 1      1
      Wrong column order                     |                 1      1
    LibPQ.Connection                         |   18            7     25
      do                                     |                 1      1
      Version Numbers                        |                 1      1
      Encoding                               |                 1      1
      Options                                |                 1      1
      Time Zone                              |                 1      1
      Finalizer                              |                 1      1
      Bad Connection                         |   18            1     19
        timeout                              |    2                   2
        closed socket (EBADF)                |                 1      1
        throw_error=false                    |    8                   8
        throw_error=true                     |    8                   8
    Results                                  |    9           14     23
      Nulls                                  |                 1      1
      Not Nulls                              |                 1      1
      Tables.jl                              |                 1      1
      Duplicate names                        |                 1      1
      Uppercase Columns                      |                 1      1
      PQResultError                          |                 1      1
      Type Conversions                       |    9            4     13
        Automatic                            |                 1      1
        Overrides                            |                 1      1
        Parsing                              |    9            2     11
          Default Types                      |                 1      1
          Specified Types                    |                 1      1
          Interval Regex                     |    9                   9
      Parameters                             |                 1      1
      Query Errors                           |                 2      2
        Syntax Errors                        |                 1      1
        Wrong No. Parameters                 |                 1      1
      Interface Errors                       |                 1      1
    Statements                               |                 2      2
      No Params, Output                      |                 1      1
      Params, Output                         |                 1      1
    AsyncResults                             |                 6      6
      Basic                                  |                 1      1
      Parameters                             |                 1      1
      Wait in line to complete               |                 1      1
      Cancel                                 |                 1      1
      Canceled by closing connection         |                 1      1
      FDWatcher: bad file descriptor (EBADF) |                 1      1
ERROR: LoadError: Some tests did not pass: 51 passed, 10 failed, 37 errored, 0 broken.
in expression starting at /home/pkgeval/.julia/packages/LibPQ/PeS3p/test/runtests.jl:32
err = ProcessFailedException(Base.Process[Process(`/opt/julia/bin/julia -Cnative -J/opt/julia/lib/julia/sys.so -g1 --code-coverage=none --color=no --compiled-modules=yes --check-bounds=yes --inline=yes --startup-file=no --track-allocation=none --eval 'append!(empty!(Base.DEPOT_PATH), ["/home/pkgeval/.julia", "/opt/julia/local/share/julia", "/opt/julia/share/julia", "/usr/local/share/julia"])
append!(empty!(Base.DL_LOAD_PATH), String[])

cd("/home/pkgeval/.julia/packages/LibPQ/PeS3p/test")
append!(empty!(ARGS), String[])
include("/home/pkgeval/.julia/packages/LibPQ/PeS3p/test/runtests.jl")
'`, ProcessExited(1))])
ERROR: Package LibPQ errored during testing
Stacktrace:
 [1] pkgerror(::String, ::Vararg{String,N} where N) at /buildworker/worker/package_linux64/build/usr/share/julia/stdlib/v1.5/Pkg/src/Types.jl:54
 [2] test(::Pkg.Types.Context, ::Array{Pkg.Types.PackageSpec,1}; coverage::Bool, julia_args::Cmd, test_args::Cmd, test_fn::Nothing) at /buildworker/worker/package_linux64/build/usr/share/julia/stdlib/v1.5/Pkg/src/Operations.jl:1471
 [3] test(::Pkg.Types.Context, ::Array{Pkg.Types.PackageSpec,1}; coverage::Bool, test_fn::Nothing, julia_args::Cmd, test_args::Cmd, kwargs::Base.Iterators.Pairs{Union{},Union{},Tuple{},NamedTuple{(),Tuple{}}}) at /buildworker/worker/package_linux64/build/usr/share/julia/stdlib/v1.5/Pkg/src/API.jl:313
 [4] test(::Pkg.Types.Context, ::Array{Pkg.Types.PackageSpec,1}) at /buildworker/worker/package_linux64/build/usr/share/julia/stdlib/v1.5/Pkg/src/API.jl:300
 [5] #test#66 at /buildworker/worker/package_linux64/build/usr/share/julia/stdlib/v1.5/Pkg/src/API.jl:294 [inlined]
 [6] test at /buildworker/worker/package_linux64/build/usr/share/julia/stdlib/v1.5/Pkg/src/API.jl:294 [inlined]
 [7] #test#65 at /buildworker/worker/package_linux64/build/usr/share/julia/stdlib/v1.5/Pkg/src/API.jl:293 [inlined]
 [8] test at /buildworker/worker/package_linux64/build/usr/share/julia/stdlib/v1.5/Pkg/src/API.jl:293 [inlined]
 [9] test(::String; kwargs::Base.Iterators.Pairs{Union{},Union{},Tuple{},NamedTuple{(),Tuple{}}}) at /buildworker/worker/package_linux64/build/usr/share/julia/stdlib/v1.5/Pkg/src/API.jl:292
 [10] test(::String) at /buildworker/worker/package_linux64/build/usr/share/julia/stdlib/v1.5/Pkg/src/API.jl:292
 [11] top-level scope at none:13
